{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Resource: DataFormats-Slides-v2.pdf\n",
    "This notebook is a high-fidelity conversion of the original PDF slide deck."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 1](rendered_slides/Formats_Full-01.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- NGS Data Formats\n\n            Dr. Jacqui Keane\n\n               @drjkeane\n           drjkeane@gmail.com\n\n\n\n\nAdapted from slides provided by Petr Danecek\n        petr.danecek@sanger.ac.uk -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 2](rendered_slides/Formats_Full-02.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Data Formats -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 3](rendered_slides/Formats_Full-03.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Illumina Sequencing Recap -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 4](rendered_slides/Formats_Full-04.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Data Formats Summary\n\n\n\n                                                                 Sequencing\n  FASTQ                                                          Instrument\n    I Unaligned read sequences with base qualities\n                                                                   FASTQ\n  SAM/BAM\n    I Unaligned or aligned reads                      Genome            Sequence\n                                                      Assembly          Alignment\n    I Text and binary formats\n  CRAM                                               FASTA                       BAM\n    I Better compression than BAM\n                                                      Genome               Variant\n  VCF/BCF                                            Annotation            Calling\n    I Flexible variant call format\n    I Arbitrary types of sequence variation          GFF                         VCF\n    I SNPs, indels, structuralvariations\n                                                      Analysis             Analysis\n  FASTA\n    I Nucleotide sequence data\n    I Reference genome, gene sequence\n  GFF\n    I\n      Genes and other features of sequences\n    I CDS, tRNA, rRNA -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 5](rendered_slides/Formats_Full-05.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- FASTA - reference genome\n\n          >1 dna:chromosome chromosome:GRCh37:1:1:249250621:1\n          NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN\n          NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN\n          NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN\n          TATTCAAAAAATTGAGAATTTCTGACCACTTAACAAACCCACAGAAAATCCACCCGAGTG\n          CACTGAGCACGCCAGAAATCAGGTGGCCTCAAAGAGCTGCTCCCACCTGAAGGAGACGCG\n          CTGCTGCTGCTGTCGTCCTGCCTGGCGCCTTGGCCTACAGGGGCCGCGGTTGAGGGTGGG\n          AGTGGGGGTGCACTGGCCAGCACCTCAGGAGCTGGGGGTGGTGGTGGGGGCGGTGGGGGT\n          GGTGTTAGTACCCCATCTTGTAGGTCTGAAACACAAAGTGTGGGGTGTCTAGGGAAGAAG\n          >2\n          NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN\n          NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN\n          NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN\n          AAAAGCATTTATGCTACAAATTACTATGGTAATTATGCTACAAATTTATGGTACCATAAA\n          TTACCATAGTAATTTGTAGCATAAATTTGTACTATGGTACAAATTACATGGGAGAGTGAA\n          GGTGGGTTAAAACATTCATATTAAAGAACTTCCACTCAGATTGCAAGAAAAGAGAGAGGA\n          ATGGAGATGGTAGCACAAGTCCCTACAATAAAAGTAGATGTTTTGAGATCAGTTCTATTT -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 6](rendered_slides/Formats_Full-06.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- FASTQ\n\n            @ERR007731.739 IL16_2979:6:1:9:1684/1           Read name\n            CTTGACGACTTGAAAAATGACGAAATCACTAAAAAACGTGAAAAATGAGAAATG...   Sequence\n        1\n       d\n            +\n      a\n   Re\n\n\n\n            BBCBCBBBBBBBABBABBBBBBBABBBBBBBBBBBBBBABAAAABBBBB=@>BB...   Base qualities\n            @ERR007731.740 IL16_2979:6:1:9:1419/1\n            AAAAAAAAAGATGTCATCAGCACATCAGAAAAGAAGGCAACTTTAAAACTTTTC...\n        2\n       d\n\n\n\n\n            +\n      a\n   Re\n\n\n\n\n            BBABB/ABABAABABABBABBBAAA>@B@BBAA@4AAA>.>BAA@779:AAA@A...\n\n\n   I    Simple format for raw unaligned sequencing reads\n   I    Paired-end sequencing: two FASTQ files or one interleaved file -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 7](rendered_slides/Formats_Full-07.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- FASTQ\n\n                @ERR007731.739 IL16_2979:6:1:9:1684/1           Read name\n                CTTGACGACTTGAAAAATGACGAAATCACTAAAAAACGTGAAAAATGAGAAATG...             Sequence\n        1\n       d\n                +\n      a\n   Re\n\n\n\n                BBCBCBBBBBBBABBABBBBBBBABBBBBBBBBBBBBBABAAAABBBBB=@>BB...             Base qualities\n                @ERR007731.740 IL16_2979:6:1:9:1419/1\n                AAAAAAAAAGATGTCATCAGCACATCAGAAAAGAAGGCAACTTTAAAACTTTTC...\n        2\n       d\n\n\n\n\n                +\n      a\n   Re\n\n\n\n\n                BBABB/ABABAABABABBABBBAAA>@B@BBAA@4AAA>.>BAA@779:AAA@A...\n\n\n   I    Simple format for raw unaligned sequencing reads\n   I    Paired-end sequencing: two FASTQ files or one interleaved file\n\n   I    Quality encoded in ASCII characters with decimal codes 33-126\n            I    ASCII code of \u201cA\u201d is 65, the corresponding quality is Q = 65\u221233 = 32\n\n                   Base quality encoded as character\n                          ! \" # $ % & ' ( ) * + , - . / 0 1 2 3 4 5 6 7 8 9 : ; < = > ? @ A B C D E F G H I J\n\n                   Numeric ASCII value\n                         33 . . . . . . . . . . . .    47 . . . . . . . . . . . . . . . . .   65 . . . . . . . .\n\n                   Base quality value                                                            (65-33 = 32)\n                          0 . . . . . . . . . . . .    14 . . . . . . . . . . . . . . . . .   32 . . . . . . . . -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 8](rendered_slides/Formats_Full-08.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Quality = Phred-scaled probability of an error\n\n\n\n                Quality    Probability of error   Accuracy\n                10 (Q10)   1 in 10                90%\n                20 (Q20)   1 in 100               99%\n                30 (Q30)   1 in 1000              99.9%\n                40 (Q40)   1 in 10000             99.99% -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 9](rendered_slides/Formats_Full-09.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- SAM / BAM: Sequence Alignment/Map format\n\n\n\n                         40M5D30M2I28M -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 10](rendered_slides/Formats_Full-10.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- SAM / BAM: Sequence Alignment/Map format\n\n\n\n\n   Flag\n      Hex     Dec    Flag            Description\n      0x1     1      PAIRED          paired-end (or multiple-segment) sequencing technology\n      0x2     2      PROPER_PAIR     each segment properly aligned according to the aligner\n      0x4     4      UNMAP           segment unmapped\n      0x8     8      MUNMAP          next segment in the template unmapped\n      0x10    16     REVERSE         SEQ is reverse complemented\n      0x20    32     MREVERSE        SEQ of the next segment in the template is reversed\n      0x40    64     READ1           the first segment in the template\n      0x80    128    READ2           the last segment in the template\n      0x100   256    SECONDARY       secondary alignment\n      0x200   512    QCFAIL          not passing quality controls\n      0x400   1024   DUP             PCR or opticalduplicate\n      0x800   2048   SUPPLEMENTARY   supplementary alignment -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 11](rendered_slides/Formats_Full-11.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- SAM / BAM: Sequence Alignment/Map format\n\n\n\n\n   Insert size\n   length of the DNA fragment sequenced from both ends by paired-end sequencing: -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 12](rendered_slides/Formats_Full-12.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- CRAM: Reference based Compression\n   BAM files are too large\n     I   ~1.5-2 bytes per base pair\n\n   Increases in disk capacity are being far outstripped by sequencing technologies\n\n\n\n\n         Zachary D. Stephens, et al, Big Data: Astronomicalor Genomical? DOI: 10.1371/journal.pbio.1002195 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 13](rendered_slides/Formats_Full-13.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- CRAM: Reference based Compression\n   BAM files are too large\n     I   ~1.5-2 bytes per base pair\n\n   Increases in disk capacity are being far outstripped by sequencing technologies\n   BAM stores allof the data\n     I   Every read base\n     I   Every base quality\n     I   Using a single conventional compression technique for all types of data\n\n                   Reference sequence: ACGTACGTACGTACGTACGTACGTACGTACGTAC\n                   read 1:             ACGTACGTACGTACGTACGTGC\n                   read 2:                TACGTACGCACGTACGTGCGTA\n                   read 3:                  CGTACGCACGTACGTACGTACG\n                   read 4:                    TACGTACGTACGTGCGTACGTA\n                   read 5:                      CGCACGTACGTACGTACGTACG\n                   read 6:                            TACGTGCGTACGTACGTAC -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 14](rendered_slides/Formats_Full-14.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- CRAM: Reference based Compression\n   BAM files are too large\n     I   ~1.5-2 bytes per base pair\n\n   Increases in disk capacity are being far outstripped by sequencing technologies\n   BAM stores allof the data\n     I   Every read base\n     I   Every base quality\n     I   Using a single conventional compression technique for all types of data\n\n                   Reference sequence: ACGTACGTACGTACGTACGTACGTACGTACGTAC\n                   read 1:             ....................G.\n                   read 2:                ........C.............\n                   read 3:                  ......C...............\n                   read 4:                    .............G........\n                   read 5:                      ..C...................\n                   read 6:                            .....G.............\n\n   CRAM: in lossless mode 60% of BAM size\n     I   Reference based compression\n     I   Controlled loss of quality information\n     I   Different compression methods for different type of data -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 15](rendered_slides/Formats_Full-15.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- VCF: Variant CallFormat\n\n\n\n\n   File format for storing variation data\n     I  tab-delimited text, parsable by standard UNIX commands\n     I  flexible and user-extensible\n     I  composed of a header section and a body section -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 16](rendered_slides/Formats_Full-16.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- VCF anatomy\n\n\n\n\n     Row-oriented, tab-delimited file with eight mandatory columns (CHROM-INFO) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 17](rendered_slides/Formats_Full-17.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- VCF anatomy\n\n\n\n\n              Genomic coordinates -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 18](rendered_slides/Formats_Full-18.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- VCF anatomy\n\n\n\n\n              Arbitrary string, typically a dbSNP RefSNP id. Dot for\n                                  missing value. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 19](rendered_slides/Formats_Full-19.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- VCF anatomy\n\n\n\n\n              Reference sequence - chr1\n\n\n\n\n                                          G   24,535\n\n\n\n\n                                          T\n                                          C   1,234,567\n\n\n\n                                          C   1,456,789 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 20](rendered_slides/Formats_Full-20.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- VCF anatomy\n\n\n\n\n              Reference sequence - chr1\n\n\n\n\n                                          G   24,535      G   A\n\n\n\n\n                                          T\n                                          C   1,234,567   G   G\n\n\n\n                                          C   1,456,789   C   T -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 21](rendered_slides/Formats_Full-21.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- VCF anatomy\n\n\n\n\n          Although in theory phred-scaled probability, don\u2019t expect\n                truly probabilistic interpretation in practice. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 22](rendered_slides/Formats_Full-22.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- VCF anatomy\n\n\n\n\n              Soft-filter variants with e.g. low quality, low depth, etc. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 23](rendered_slides/Formats_Full-23.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- VCF anatomy\n\n\n\n\n                Per-site annotations across all samples\n          Here DP is the cumulative read depth across all samples -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 24](rendered_slides/Formats_Full-24.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- VCF anatomy\n    anatomy\n\n\n\n\n                Per-site\n           Per-sample      annotations\n                        annotations.     across\n                                      Here        all samples\n                                            GT (genotype)  and AD\n          Here DP is the cumulative read depth across all samples\n               (allelic depth) will be present for each sample.\n\n\n\n\n                                                                    22 / 48 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 25](rendered_slides/Formats_Full-25.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- VCF anatomy\n    anatomy\n\n\n\n\n          Per-sample values listed in the same order as specified in\n               the FORMAT column, separated by a colon. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 26](rendered_slides/Formats_Full-26.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- VCF anatomy\n    anatomy\n\n\n\n\n                 Per-site\n           Per-sample       annotations\n                       values listed  in the across\n                                              same    all samples\n                                                   order  as\n             Multiple\n          Here  DP     alternate  alleles canread\n                                               be present  in specified\n                                                              one row. in\n                 theisFORMAT\n                        the cumulative\n                                  column, separateddepth\n                                                       by aacross\n                                                              colon. all samples\n\n\n\n\n                                                                                   22 / 48 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 27](rendered_slides/Formats_Full-27.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- VCF vs BCF\n\n   VCFs can be very big\n    I compressed VCF with 3781 samples, human data:\n          I   54 GB for chromosome 1\n          I   680 GB whole genome\n   VCFs can be slow to parse\n    I text conversion is slow\n    I main bottleneck: FORMAT fields\n\n\n\n\n   BCF\n    I    binary representation of VCF\n    I    fields rearranged for fast access -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 28](rendered_slides/Formats_Full-28.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Data Formats Summary -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 29](rendered_slides/Formats_Full-29.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Data Formats Summary -->"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "name": "python",
   "version": "3.8.10"
  },
  "celltoolbar": "Slideshow"
 },
 "nbformat": 4,
 "nbformat_minor": 4
}