{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Resource: MQ Sequencing Tech 2025.pdf\n",
    "This notebook is a high-fidelity conversion of the original PDF slide deck."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 1](rendered_slides/MQ_Tech_Full-001.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Sequencing Technologies\n\n       Michael Quail\n       mq1@sanger.ac.uk -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 2](rendered_slides/MQ_Tech_Full-002.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Short Read\n                Sequencers\n                                              Illumina\n                                              Nextseq\n\nOmniome\n\n\n\n\n          Element   Ultima UG100\n                                   Singular\n                                   S4 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 3](rendered_slides/MQ_Tech_Full-003.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Long Read Sequencers\n\n\n                                  Oxford Nanopore P48\n\n    PacBio Revio\n                                                                     Oxford Nanopore gridION\n\n\n\n\n                                            Oxford Nanopore minION\n\n                    PacBio Vega\n\n\n\nGenerate much longer reads that enable more of the genome to be covered\nGenerally more expensive than short read, with higher input requirements,\nSingle molecule so yields dependent on DNA quality -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 4](rendered_slides/MQ_Tech_Full-004.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Sanger Sequencing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 5](rendered_slides/MQ_Tech_Full-005.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Frederick Sanger\n\n\u00bb Discovered DNA sequencing by chain\n  termination method\n\u00bb Nobel Prize 1 (1958)\n  \u00bb Complete amino acid\n    sequence of insulin\n\u00bb Nobel Prize 2 (1980)\n  \u00bb For DNA sequencing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 6](rendered_slides/MQ_Tech_Full-006.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Primer Extension\n                             DNA Polymerase\n\n\n           Primer\n\n               dATP\n               dGTP\n               dCTP\n               dGTP\n\n               Nucleotides       NEW\n                                 STRAND\n\n\n\n\nTemplate DNA -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 7](rendered_slides/MQ_Tech_Full-007.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Dideoxy Nucleotides\n\n\n\u2022 Lack an -OH group at the 3-\n    carbon position\n\u2022   Cannot add another\n    nucleoside at that position\n\u2022   Prevent further DNA\n    synthesis -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 8](rendered_slides/MQ_Tech_Full-008.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- All Possible Terminations -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 9](rendered_slides/MQ_Tech_Full-009.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Original Sanger Sequencing\n\u00bb 4 sequencing reactions performed for each\n  template, each with different terminator\n\u00bb Radioactive primer or nucleotide used\n\u00bb Sequencing reactions run on <1mm\n  polyacrylamide gel cast between two glass\n  plates to separate fragments according to\n  size\n\u00bb After run gel exposed to film and\n  developed to reveal image -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 10](rendered_slides/MQ_Tech_Full-010.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Sequencing gel autorad -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 11](rendered_slides/MQ_Tech_Full-011.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Fluorescent Terminators -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 12](rendered_slides/MQ_Tech_Full-012.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Fluorescent Gel Sequencing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 13](rendered_slides/MQ_Tech_Full-013.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- ABI Capillary 3730 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 14](rendered_slides/MQ_Tech_Full-014.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- DNA Sequence Files -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 15](rendered_slides/MQ_Tech_Full-015.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Error\n\n\u00bb Sequence quality Q is reported on a log scale\n\u00bb Q10 is 1 error in 10\n\u00bb Q20 is 1 error in 100\n\u00bb Q30 is 1 error in 1000\n\u00bb Q40 is 1 error in 10000\n\u00bb Q50 is 1 error in 100000 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 16](rendered_slides/MQ_Tech_Full-016.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Capillary sequencing\n\u00bb Individual reactions -> 96 capillary array\n\u00bb PCR errors\n\u00bb Cloning bias\n\u00bb 1000-base reads\n\u00bb 1-2 hour run time\n\u00bb Accurate, Q30\n\u00bb Low cost if want 1 read\n\u00bb Low throughput, expensive to sequence\nwhole genomes -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 17](rendered_slides/MQ_Tech_Full-017.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Next Generation sequencing\n\n       Is massively parallel\nNot limited to few reactions per run -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 18](rendered_slides/MQ_Tech_Full-018.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Next-Generation Sequencing\n\n\n\n\n1 feature         1 chip,thousands or\n1 template        millions of features\n                  Output Mb-Tb -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 19](rendered_slides/MQ_Tech_Full-019.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Library prep\n\n\n\n\n\u2022 Library -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 20](rendered_slides/MQ_Tech_Full-020.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- \u2022 Library -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 21](rendered_slides/MQ_Tech_Full-021.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- \u2022 Illumina Library\n\n\n\n\n  P5\n\n\n\n                       R1       R2                    P7\n  Flowcell annealing\n                       Primer   Primer site\n                       site                   Flowcell annealing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 22](rendered_slides/MQ_Tech_Full-022.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- \u2022 Illumina Library\n\n\n\n\n  P5                   i5                      i7\n\n\n\n                            R1       R2                     P7\n  Flowcell annealing\n                            Primer   Primer site\n                            site                    Flowcell annealing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 23](rendered_slides/MQ_Tech_Full-023.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Library Prep Approaches\n\n1. Classical adapter ligation method\n\n2. Transposon mediated\n\n3. Inclusion of adapter sequences during\n   PCR -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 24](rendered_slides/MQ_Tech_Full-024.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Illumina Paired End\n5\u2019\n                                Library Prep\n                        P\n\n\n          T                 5\u2019                         A\n          P\n                    +       A                          5\u2019\n                                                            P\n\n\n\n3\u2019                                       T4 DNA Ligase\n                                                                                     R1 sequencing primer\n\n              5\u2019                                                      3\u2019\n\n\n                                 T                       A\n                                                         T\n                             A\n\n                                                                                     Reverse compliment of R2 primer\n                                                                      5\u2019\n              3\u2019\n\n                   P5                    Hybridize primers\n\n\n              5\u2019                                                      3\u2019   P7\n\n\n                             T                           A\n                                                         T\n                             A\n\n\n                                                                      5\u2019\n              3\u2019\n                                         Limited PCR\n\n\n\n\n     5\u2019                              T                      A                   3\u2019\n     3\u2019                              A                      T                   5\u2019\n\n                                         Make clusters and sequence -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 25](rendered_slides/MQ_Tech_Full-025.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Nextera -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 26](rendered_slides/MQ_Tech_Full-026.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Amplicon sequencing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 27](rendered_slides/MQ_Tech_Full-027.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- 454 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 28](rendered_slides/MQ_Tech_Full-028.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- 454\n\n\u00bb Started NGS (2005)\n\u00bb First massively parallel sequencer\n\u00bb Bought by Roche in 2007. Now\n  discontinued\n\u00bb Based on pyrosequencing of bead-\n  bound DNA in microwells\n\u00bb Fore-runner of Ion Torrent and\n  Genapsys -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 29](rendered_slides/MQ_Tech_Full-029.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- 454/Roche Summary\n\n\u00bb .7 Gb / run\n\u00bb 700 base reads\n\u00bb <24 hour run time\n\u00bb $7,000 / Gb\n\n\n\nRoche discontinued in 2016 \u2013 Too expensive -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 30](rendered_slides/MQ_Tech_Full-030.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Pyrosequencing\n\n\n\n\n\u00bb The incorporation of new bases releases inorganic pyrophosphate\n\n \u00bb A chemical cascade converts luciferin to oxy-luciferin + light -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 31](rendered_slides/MQ_Tech_Full-031.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Depositing DNA Beads into the PicoTiter\u2122Plate\n\n                                   Load Enzyme Beads\n\nLoad beads into\nPicoTiter\u2122Plate\n\n\n\n\n                                   Centrifugation\n\n\n\n\n                  44 \u03bcm -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 32](rendered_slides/MQ_Tech_Full-032.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- 454 Technology - Sequencing Instrument -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 33](rendered_slides/MQ_Tech_Full-033.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- 454 Data Example -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 34](rendered_slides/MQ_Tech_Full-034.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- 454/Roche Summary\n\u00bb Long read lengths \u2013 good for amplicons and de-novo\nsequencing\n\u00bb High error rate near homopolymers\n\u00bb Single end only\n\u00bb .7 Gb / run\n\u00bb 700-1000 base reads\n\u00bb <24 hour run time\n\u00bb $7,000 / Gb\n\n\n\nRoche discontinued in 2016 \u2013 Too expensive -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 35](rendered_slides/MQ_Tech_Full-035.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Other Flow technologies\n\n\u00bb Ion Torrent. As 454 but detects the H+\n  released as a base is incorporated.\n\n\n\n\u00bb Genapsys. As 454 but detects increase in\n  impedence as a base is incorporated. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 36](rendered_slides/MQ_Tech_Full-036.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Ion Torrent\u2019s PGM\n\n\n\n\nCapital cost $50,000 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 37](rendered_slides/MQ_Tech_Full-037.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Ion Torrent\n\u00bb Similar to 454 but detects H+ released as a base is\n  added\n\n\u00bb Prone to errors near homopolymers\n\n\u00bb Not good for whole genome sequencing but useful\n  for targeted sequencing as run times are short\n\n\u00bb Used in a lot of clinical settings for disease panel\n  sequencing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 38](rendered_slides/MQ_Tech_Full-038.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Ion Torrent\u2019s Technology\n                   DNA\n                polymerase\n                nucleotides\n                                                well\n                                            ion-sensing layer\n                                            electronics layer\n\n   DNA(n)+ nucl. = DNA(n+1) + PPi -> PPi + H+\n\n      Flow through A then T then ... like 454 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 39](rendered_slides/MQ_Tech_Full-039.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Ion Torrent Chip Drawing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 40](rendered_slides/MQ_Tech_Full-040.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Ion Torrent\n\u00bb Library prep like original 454\n\u00bb Amplification on beads by emPCR\n\u00bb CMOS chip detection\n\u00bb Cyclic addition sequencing - pH changes\n\u00bb Not single-molecule -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 41](rendered_slides/MQ_Tech_Full-041.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 42](rendered_slides/MQ_Tech_Full-042.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Ion AmpliSeq On-Demand Panels | Now 5,000 Pre-Designed, Pre-Tested Genes Available\n\n                         Expanded Gene Content Across Disease Research Areas\n                  2500\n\n                  2000\n     # of Genes\n\n\n\n\n                  1500\n\n                  1000\n\n                   500\n\n                     0\n\n\n\n\n                         Average of 1154 genes per major UMLS disease research area category\n                                        For Research Use Only. Not for use in diagnostic procedures.\n\n\n\n\n42 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 43](rendered_slides/MQ_Tech_Full-043.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Ion GeneStudio S5 Series | Flexible Portfolio Configurable to Your Needs\n\n\n\n\n              Ion GeneStudio\u2122 S5                                                     Ion GeneStudio\u2122 S5 Plus              Ion GeneStudio\u2122 S5 Prime\n\n                                                                                                       New                       New\n\n\n\n\n                             Fast.                                                                Flexible.                    Powerful\n                                                                                                                                  .\n                                                                                                                                            New\n\n\n\n\n      Ion 510\u2122                                  Ion 520\u2122                                 Ion 530\u2122 Chip         Ion 540\u2122 Chip            Ion 550\u2122 Chip\n         Chip                                      Chip                                  15\u201320 M reads         60\u201380 M reads           100\u2013130 M reads\n     2\u20133 M reads                               3\u20136 M reads                                Up to 600 bp          Up to 200 bp             Up to 200 bp\n     Up to 400 bp                              Up to 600 bp\n     For Research Use Only. Not for use in diagnostic procedures. * Throughputs based on 200bp sequencing\n\n\n\n\n43 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 44](rendered_slides/MQ_Tech_Full-044.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Output and Turn-Around Time to Meet Your Lab\u2019s Peak Volume Needs\n\n\n\n\n                                                                Ion GeneStudio\u2122 S5       Ion GeneStudio\u2122 S5 Plus   Ion GeneStudio\u2122 S5 Prime\n\n\n\n\n                  Speed*                                                      19 hrs              10 hrs                    6.5 hrs\n\n\n        Output (max/day):                                                   15 Gb/80 M        30 Gb/160 M                50 Gb/260 M\n\n\n         Chips (max/day):                                                    1 x 540        2 x 540 or 1 x 550             2 x 550\n\n * Based off 540 chip \u2013 sequencing (2.5 hours) and analysis (varies) time\n\n\n\n     For Research Use Only. Not for use in diagnostic procedures.\n\n\n\n\n44 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 45](rendered_slides/MQ_Tech_Full-045.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- $299k\n     12-15M reads\n\n\n\n\n45 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 46](rendered_slides/MQ_Tech_Full-046.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- 46 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 47](rendered_slides/MQ_Tech_Full-047.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Sequencing by Synthesis\n\nwith florescence detection -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 48](rendered_slides/MQ_Tech_Full-048.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Illumina -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 49](rendered_slides/MQ_Tech_Full-049.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Illumina Library Insert sizes\n          Exome, targeted, ChiP, ATAC\n     P5               SP 1          100-200bp\n\n\n                                                 SP2                         P7\n                      Primer site\n                                                 Primer site\n\nReads need to be just long enough to map and ideally not to overlap exon boundaries\n\n\n\n\n    Whole Genome\n     P5               SP 1           400-600bp                                    P7\n\n\n                                                               SP2\n                      Primer site\n                                                               Primer site\n\n    Fragments need to be just long to span common repeat elements eg AluI\n    with unique sequence on either side so can map -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 50](rendered_slides/MQ_Tech_Full-050.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 51](rendered_slides/MQ_Tech_Full-051.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 52](rendered_slides/MQ_Tech_Full-052.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- iSeq100, HiSeq4000, Xten, NovaSeq -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 53](rendered_slides/MQ_Tech_Full-053.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Since 2016 Illumina flowcells have been patterned\nwith clusters formed in a defined array of wells -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 54](rendered_slides/MQ_Tech_Full-054.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 55](rendered_slides/MQ_Tech_Full-055.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- ExAmp -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 56](rendered_slides/MQ_Tech_Full-056.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 57](rendered_slides/MQ_Tech_Full-057.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 58](rendered_slides/MQ_Tech_Full-058.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 59](rendered_slides/MQ_Tech_Full-059.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 60](rendered_slides/MQ_Tech_Full-060.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Sequencing by Synthesis\n\n         \u00bb Extend by 1 base\n\n         \u00bb Image\n\n\n         \u00bb Reverse\n           termination\n\n\n         \u00bb Repeat -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 61](rendered_slides/MQ_Tech_Full-061.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Illumina Modified Nucleotides\n                                 PPP        5\u2019       Base\n                 Natural dNTP:                   O\n\n\n\n\n                                       3\u2019   OH\n\n\n\n\nReversible florescent dTTP: -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 62](rendered_slides/MQ_Tech_Full-062.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 63](rendered_slides/MQ_Tech_Full-063.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 64](rendered_slides/MQ_Tech_Full-064.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 65](rendered_slides/MQ_Tech_Full-065.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Error\n\n\u00bb Sequence quality Q is reported on a log scale\n\u00bb Q10 is 1 error in 10\n\u00bb Q20 is 1 error in 100\n\u00bb Q30 is 1 error in 1000\n\u00bb Q40 is 1 error in 10000\n\u00bb Q50 is 1 error in 100000 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 66](rendered_slides/MQ_Tech_Full-066.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Error limits read length -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 67](rendered_slides/MQ_Tech_Full-067.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Illumina sequencing\n         with indexing\nWith Illumina sequencing you sequence the insert from\n   each end. And you can also perform sequencing\n reactions that read the sequence of the index in the\n        introduced adapter. So how is this done -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 68](rendered_slides/MQ_Tech_Full-068.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 69](rendered_slides/MQ_Tech_Full-069.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Economics of Illumina \u2019s Sequencing Portfolio\n\n\nPlatform       Flow Cell   $/G                              $/G\n                           (300 CYC.)\n                                                          (300 CYC.)\n\n\n               10B         $3\nNovaseq X/X+\n               25B         $2\n\n               S4          $5\nNovaSeq 6000\n               SP          $15.40\n\n\nNextSeq 2000   P3          $20.00\n\n\nNextSeq\n               P2          $29.50\n1000/2000\n\n\nNextSeq 550    HO          $41.38\n\n\n\nNextSeq 550    MO          $47.50\n\n\n\nMiSeq          v2          $236.00\n\n\n                                        $0   $50   $100       $150     $200 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 70](rendered_slides/MQ_Tech_Full-070.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 71](rendered_slides/MQ_Tech_Full-071.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- MiSeq i100 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 72](rendered_slides/MQ_Tech_Full-072.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Illumina\n\n\u00bb Cheap $2-$300/Gb\n\u00bb Highly accurate data mostly Q30\n\u00bb Massively parallel. Millions/billions of\n  reads\n\u00bb Short read -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 73](rendered_slides/MQ_Tech_Full-073.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- The ownership of this material belongs to MGI\uff0cand without the permission of the owner, it shall not be directly release\nor disclosed.\n\n\nMGI All Rights Reserved Dissemination\uff0cdistribution and copying is prohibited without authorization. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 74](rendered_slides/MQ_Tech_Full-074.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 75](rendered_slides/MQ_Tech_Full-075.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 76](rendered_slides/MQ_Tech_Full-076.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 77](rendered_slides/MQ_Tech_Full-077.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- MGI sequencing chemistry\nDNA nanoballs and cool MPS -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 78](rendered_slides/MQ_Tech_Full-078.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 79](rendered_slides/MQ_Tech_Full-079.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 80](rendered_slides/MQ_Tech_Full-080.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- DNBseq-T1+\n\n\n\n\n12Tb in 22 hours\nApprox $2/Gb\nUpto PE300 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 81](rendered_slides/MQ_Tech_Full-081.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- New\n           Sequencers\n                                           Illumina\n                                           Chemistry X\n\nOnso\n\n\n\n                 Ultima UG100\n\n\n\n\n       Element\n\n                                      Singular\n                                      S4\n\n\n                                MGI -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 82](rendered_slides/MQ_Tech_Full-082.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Comparison Table\n                 Illumina        Illumina         Ultima UG100    Element Aviti       Singular G4       PacBio SBB\n                 Novaseq         Nextseq\n\nInstrument       $1.25M          $335k            $1.5M           $289k ($249K)       $350k             $99K\n\n$/Gb             3 (10B), 2      20 (9 in 2024)   1               2-5                 8-10              4-12\n                 (25B)\n\nAccuracy         QQQ             QQQ              QQQ             QQQQ                QQQ               QQQQQ\n\nMax read         2x150 (SP)      2x300            300 SE          2x300               2x150             200SE\nlength\n\nRun Time for     24 hrs          48 hrs           18 hrs          48 hrs              19 hrs            48hrs\n2x150 or 1x300\n\nFlowcells        2               2                2               2                   4                 2\n\nYield/run        6000Gb,         30-360Gb         >4000Gb         480Gb               360Gb             150Gb\n                 16000Gb\n\nOther            Infinity long   Infinity long                    Non patterned,                        Non patterned\n                 reads           reads                            loop long reads\n\nUse case                                          Cheap           -high plex levels   Flexibility via   High accuracy\n                                                  sequence for    \u2013low                independent       applications,\n                                                  big data        complexity          flowcells         high plex\n                                                  applications,   -long reads                           potential -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 83](rendered_slides/MQ_Tech_Full-083.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Element AVITI\u2122\n\u25cf ~$335K. $249K each if you order 3\n\n\u25cf $2-$5/Gb. $1680 for 200-300Gb\n\n\u25cf 2 flowcells. 2 x 150. 90% >Q40\n\n\u25cf 2 x 300 available\n\n\u25cf Sequencing by binding on RCA amplified template\n\n\u25cf Loop synthetic long read\n\n\u25cf Low duplicates\n\n\u25cf No index hopping allowing for combinatorial indexing\n\n\u25cf Better results with low complexity sequencing\n\n\u25cf Can have long inserts >2kb\n\n\n\n\nImages provided by Element Biosciences -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 84](rendered_slides/MQ_Tech_Full-084.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Element Kits -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 85](rendered_slides/MQ_Tech_Full-085.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Avidity base chemistry (ABC)\n\n\n\n\nContent and Images provided by Element Biosciences -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 86](rendered_slides/MQ_Tech_Full-086.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- AVITI System Architecture Enables\nMaximum Run Flexibility\n   Dual Independent Flow Cells       Individually Addressible Lanes\n\n\n\n\n    Perform two parallel runs with   Sequence separate libraries or\n      different run parameters         pools on a single flow cell -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 87](rendered_slides/MQ_Tech_Full-087.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Cloudbreak Freestyle\n\n\n\n\nContent and Images provided by Element Biosciences -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 88](rendered_slides/MQ_Tech_Full-088.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Achieving Q50 with UltraQ\n\n\n\n\nContent and Images provided by Element Biosciences -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 89](rendered_slides/MQ_Tech_Full-089.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Trinity: On Flow cell\n                hybrid capture\n\n\n\n\nContent and Images provided by Element Biosciences -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 90](rendered_slides/MQ_Tech_Full-090.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Expert Mode HD: 20-70% more data\nfor FREE with higher polony density\n\n\n\n\nContent and Images provided by Element Biosciences -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 91](rendered_slides/MQ_Tech_Full-091.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Singular Genomics\n\n\n\u25cf$10-$15/Gb\n\u25cf2 x 150 in 19 hours\n\u25cf4 x 4 lane flowcells. Each flowcell can be run and loaded\n  independently.\n\u25cfUses similar SBS and clustering to Illumina\n\u25cfNo index hopping. No issue with low complexity -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 92](rendered_slides/MQ_Tech_Full-092.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- 2\n\n\n\n\n    Ultima -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 93](rendered_slides/MQ_Tech_Full-093.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Ultima\n\u2022 $100 genome. $1/Gb\n\u2022 SE 300 in ~14hr\n\u2022 Flow based.\n\u2022 Some indel errors around homopolymers\n\u2022 89% Q30\n\u2022 Flow based sequencing on a 10\u201d spinning disc -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 94](rendered_slides/MQ_Tech_Full-094.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Revolutionary new sequencing hardware\n\n                            Camera\n\n\n\n\n        Reagent dispenser\n\n                                                    Spinning wafer on instrument\n                                                                                   \u2022   Low cost (standard substrate)\n\nWafer\n                                                                                   \u2022   ~10B reads per wafer (x2)\n\n\n                                                                                   \u2022   Fast and efficient reagent delivery\n\n\n                                                                                   \u2022   \u201cAir gap\u201d minimized contamination\n\n\n                                                                                   \u2022   Rotational optical scanning\n\n\n\n\n                                                                                                                             Proprietary and Confidential -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 95](rendered_slides/MQ_Tech_Full-095.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- A novel spin on sequencing chemistry\n                                                 mnSBS\n\n  Flow Chemistry                           Terminated chemistry   Mostly natural sequencing by synthesis\n\n                      Diffusion\n\n\n\n\n                                                                                                            Labeled\n\n\n\n\n                                                                                                                 Unlabeled\n\n\n\n\nNatural nucleotides                      Reversible terminator     Fluorescently labeled                      Mostly natural\n\n\n\n\n                                                                   Long read length\n                                                                   Fast cycle time\n                                                                   Accurate\n                                                                   Scalable\n                                                                   Low cost\nTransient detection                      End point detection\n\nStatic sensor                            Scanning\n\n\n\n\n                                                                                                           Proprietary and Confidential -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 96](rendered_slides/MQ_Tech_Full-096.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Mostly natural chemistry combines advantages of\n           flow chemistry with optical scanning\n                                                      Median signals vs. homopolymer\n\n\n\n\n                                                                          homopolymer level\n\n\n\n\n\u2022 Majority nucleotides unlabeled to avoid quenching   \u2022 Endpoint detection significantly improves accuracy\n\u2022 Minimal scarring supports longer reads              \u2022 Maintain signal linearity to at least homopolymer length 12\n\u2022 Faster runs via 2min wash->image->cleave cycle      \u2022 Machine learning accounts for sequence context -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 97](rendered_slides/MQ_Tech_Full-097.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Mostly natural chemistry combines advantages of\n               flow chemistry with optical scanning\n                                                        Median signals vs. homopolymer\n\n\n\n\n                                                                                     homopolymer level\n\n\n\n\u2022   Majority nucleotides unlabeled to avoid quenching   \u2022   Endpoint detection significantly improves accuracy\n\u2022   Minimal scarring supports longer reads              \u2022   Maintain signal linearity to at least homopolymer length 12\n\u2022   Faster runs via 2min wash->image->cleave cycle      \u2022   Machine learning accounts for sequence context\n\n\n\n\n                                                                                                                          Proprietary and Confidential -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 98](rendered_slides/MQ_Tech_Full-098.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Mostly natural chemistry combines advantages of\n                flow chemistry with optical scanning\n                                                        Median signals vs. homopolymer\n\n\n\n\n                                                                                     homopolymer level\n\n\n\n\u2022   Majority nucleotides unlabeled to avoid quenching   \u2022   Endpoint detection significantly improves accuracy\n\u2022   Minimal scarring supports longer reads              \u2022   Maintain signal linearity to at least homopolymer length 12\n\u2022   Faster runs via 2min wash->image->cleave cycle      \u2022   Machine learning accounts for sequence context\n\n\n\n\n                                                                                                                          Proprietary and Confidential -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 99](rendered_slides/MQ_Tech_Full-099.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Mostly natural chemistry combines advantages of\n               flow chemistry with optical scanning\n                                                        Median signals vs. homopolymer\n\n\n\n\n                                                                              homopolymer level\n\n\n\n\n\u2022   Majority nucleotides unlabeled to avoid quenching   \u2022   Endpoint detection significantly improves accuracy\n\u2022   Minimal scarring supports longer reads              \u2022   Maintain signal linearity to at least homopolymer length 12\n\u2022   Faster runs via 2min wash->image->cleave cycle      \u2022   Machine learning accounts for sequence context\n\n\n                                                                                                                          Proprietary and Confidential -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 100](rendered_slides/MQ_Tech_Full-100.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 101](rendered_slides/MQ_Tech_Full-101.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- PPM seq \u2013 1 in a million VAF\n        accuracy -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 102](rendered_slides/MQ_Tech_Full-102.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- PPM seq- library prep -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 103](rendered_slides/MQ_Tech_Full-103.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Roche SBX\n\n- Nanopore sequencer enabled\n by acqusition of Genia and Stratos\n\n- Launch 2026\n- 500Mb/sec\n- 15B reads in 4 hours\n- 7B in 1 hour \u2013 fast mode\n- Q39 for duplex libraries (200-\n  300bp)\n- Q23 for simplex (200-1500bp)\n\n- Instrument $400-$600k?\n\n- Consumable pricing = competitive\n\n- Read the paper\nDOI: 10.1101/2025.02.19.639056 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 104](rendered_slides/MQ_Tech_Full-104.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 105](rendered_slides/MQ_Tech_Full-105.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 106](rendered_slides/MQ_Tech_Full-106.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Libraries are converted to expandomers\n  Bases copied as XNTPs (50x bigger) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 107](rendered_slides/MQ_Tech_Full-107.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 108](rendered_slides/MQ_Tech_Full-108.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 109](rendered_slides/MQ_Tech_Full-109.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 110](rendered_slides/MQ_Tech_Full-110.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 111](rendered_slides/MQ_Tech_Full-111.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Long Read Sequencers\n\n\n\n               Oxford Nanopore flongle        Oxford Nanopore minION         Oxford Nanopore gridION\n\n\n\n\nPacBio Revio\n\n\n\n                    Oxford Nanopore P2 Solo\n                                                       Oxford Nanopore P48 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 112](rendered_slides/MQ_Tech_Full-112.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Why Long Read\n\n\u00bb Potential for higher diagnostic rate\n\n\u00bb More complete coverage of genome\n\n\u00bb Better assembly\n\n\u00bb Haplotype linkage\n\n\u00bb Simultaneous modified base detection -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 113](rendered_slides/MQ_Tech_Full-113.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Long Read Platform Comparison -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 114](rendered_slides/MQ_Tech_Full-114.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Pacific Biosciences\n\n\n\n\nOnso                     Revio\n\u00bb Short Read sequencer   \u00bb Long Read sequencer -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 115](rendered_slides/MQ_Tech_Full-115.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- PacBio Long Read\n\n\u00bb Long Read Platforms\n  RS>RSII>Sequel>SequelII>Revio\n\n\u00bb SMRT. Single Molecule Real Time\n\n\u00bb 3rd generation? -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 116](rendered_slides/MQ_Tech_Full-116.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Pacific Biosciences\nSequel II/IIe (2018-2022)\n         \u00bb 8 million ZMWs/SMRT\n         \u00bb $495K/$525 instrument -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 117](rendered_slides/MQ_Tech_Full-117.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Learn more:\n\n\n\n\n\u00bb Revio system\nThroughput\n\n\n\n            480 Gb                                                  24 hour\n            HiFi yield per run with SPRQ                            Sequencing time\n\n\n\n\n            1,300 30x genomes / year\n            2600 per year ta 20x\n\n\n\n\n                                                                                      List price, USD\n                                                                                                        $995/genome at 30x*\n   * $995 for sequencing reagents for one Revio SMRT Cell, which has an\n                                                                                      $599,000\n                                                                                                        <$500 at 20x\n   expected yield of 90 Gb, equivalent to a 30\u00d7 human genome. Expected\n   pricing subject to change, your local sales representative can provide\n   detailed pricing in your currency.\n                                                                                                                              117 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 118](rendered_slides/MQ_Tech_Full-118.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Nov\u2019 24 Low Throughput Benchtop -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 119](rendered_slides/MQ_Tech_Full-119.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Pac Bio Technology\n\n\n\n\n       Individual ZMW                Laser light illuminates the ZMW\n\n\n\n\nZMW with DNA polymerase               ZMW with polymerase + nucs. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 120](rendered_slides/MQ_Tech_Full-120.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- PacBio Template Preparation\n\n\n\n\n1   Anneal primer\n\n2   Bind polymerase\n\n3   Sequence -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 121](rendered_slides/MQ_Tech_Full-121.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- PacBio Sequencing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 122](rendered_slides/MQ_Tech_Full-122.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- PacBio Sequencing\n\n\n     \u00bb Some bases added very\n       quickly and missed\n     \u00bb Some wrong bases flirt with\n       active site and go away\n     \u00bb SMRT cell has millions of\n       ZMWs (8M Sequel, 25M\n       Revio) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 123](rendered_slides/MQ_Tech_Full-123.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Revio Polymerase Read Lengths -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 124](rendered_slides/MQ_Tech_Full-124.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 125](rendered_slides/MQ_Tech_Full-125.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Accuracy -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 126](rendered_slides/MQ_Tech_Full-126.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Long read can access 193 medically\nrelevant genes that short read cannot -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 127](rendered_slides/MQ_Tech_Full-127.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 128](rendered_slides/MQ_Tech_Full-128.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Epigenetics -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 129](rendered_slides/MQ_Tech_Full-129.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Base Modifications -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 130](rendered_slides/MQ_Tech_Full-130.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- DETECTION OF DNA BASE MODIFICATIONS USING KINETICS\n\n            Example: N6-methyladenine\n\n\n\n\n\u2022   SMRT Sequencing uses kinetic information from each nucleotide addition to call bases\n\u2022   This same information can be used to distinguish modified and native bases by comparing results of SMRT Sequencing to an in silico\n    kinetic reference for incorporation dynamics without modifications.\n\n\n\n\n                                                                                           Flusberg et al. (2010) Nature Methods 7: 461-465 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 131](rendered_slides/MQ_Tech_Full-131.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Revio variant calling and                                          Accuracy, F1%\n\nmethylation performance        100\n                                            99.95     99.95            99.41     99.44\n                                                                                                  95.19     95.59\n\n                                 80\n\n\n\n                                 60\n\n\n\n                                 40\n\n\n\n                                 20\n  +5mC\n\n\n\n\n                                   0\n                                                SNVs                       Indels                       SVs\n\n                                                       Sequel IIe system            Revio system\n\n\n             PEG3\n                            HG002 at maternally imprinted PEG3 locus\n                            Data for internal beta testing https://downloads.pacbcloud.com/public/revio/2022Q4/HG002-\n                            rep3/analysis/HG002.m84005_220827_014912_s1.GRCh38.bam\n                            Variant calling measured against Genome in a Bottle benchmarks, HG002                       131 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 132](rendered_slides/MQ_Tech_Full-132.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- MAS-Seq: high-throughput, full-length single-cell sequencing\n\n\n\n\n                           Full-length, 10x\n                          single-cell cDNA\n\n\n\n\n                                                  MAS-Seq for 10x Single Cell 3\u2019 kit\n                                                          (102-659-600)\n\n                                                      \u2022 16x concatenation\n                             A                                                                             Q\n                                              B                 C        N                O            P\n                                       B\u2019              C\u2019           D\u2019           O\u2019               P\u2019\n\n\n\n\n                                        A         B         C       D        N        O       P        Q\n\n\n\n\n https://www.biorxiv.org/content/10.1101/2021.10.01.462818v1 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 133](rendered_slides/MQ_Tech_Full-133.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Cell atlas, single-cell sequencing\n\n\n\n                                                                                                            3\n                                                        scRNA-Seq\n\n\n\n\n                            scIso-Seq\n\n\n      of human multi-exon                                                                                                          of brain expressed\n                                                                          isoforms per gene\n95%   genes have more                                      >7             on average2                                    2,900     genes are heritable\n      than one isoform1                                                                                                            at isoform level4\n\n\n\n\n      1https://www.sciencedirect.com/science/article/pii/S2001037020305341;\n      2https://www.biorxiv.org/content/10.1101/2022.06.08.495354v1.full; 3https://www.pnas.org/doi/full/10.1073/pnas.2114326118;\n      4https://www.medrxiv.org/content/10.1101/2022.10.18.22281204v1 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 134](rendered_slides/MQ_Tech_Full-134.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Comparison of PacBio with other\n         approaches -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 135](rendered_slides/MQ_Tech_Full-135.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Pacific BioSciences\n   Sequel/Revio Applications\n\u00bb Long read applications\n\u00bb De Novo sequencing\n\u00bb Full length cDNA sequenicng\n\u00bb Haplotyping\n\u00bb Variant calling with access to repetitive regions\n\n\n\u00bb DNA modification studies -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 136](rendered_slides/MQ_Tech_Full-136.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Pacific Biosciences\n\n\u00bb Single polymerase mol. in a 20nm hole\n\u00bb Watch incorporation in real time\n\u00bb ~2 bases per second\n\u00bb Yield 120Gb\n\u00bb ~$5-10/Gb\n\u00bb HiFi reads 15-20kb -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 137](rendered_slides/MQ_Tech_Full-137.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- PacBio Onso\n\u25cf   Released 2023\n\u25cf   Most accurate short read\n\u25cf   90%> Q40, Q50 possible\n\u25cf   $99K current offer price\n\u25cf   $4-12/Gb\n\u25cf   100Gb/run\n\u25cf   1x200 or 2 x 150 in 48hr\n\u25cf   Sequencing by binding\n\u25cf   Accuracy to 0.001% VAF\n    without UMIs -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 138](rendered_slides/MQ_Tech_Full-138.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Sequencing by Binding -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 139](rendered_slides/MQ_Tech_Full-139.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- SNV and Indel accuracy are better than any other\nplatform -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 140](rendered_slides/MQ_Tech_Full-140.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- SBB demonstrates >4\u00d7                                 100%\n                                                                                 1.4x\n                                                                               sensitivity\nimprovement in sequencing                            90%\n\nefficiency                                           80%\n\n                                                     70%\n\n                                                     60%\n\n\n\n\n                                       Sensitivity\n                                                     50%\n\n                                                     40%\n       6,000\u00d7 non-UMI SBB sequencing\n       exceeds >24,000\u00d7 SBS UMI                      30%\n       sequencing at 0.05% and 0.1%                                                                             SBB 6,000\u00d7\n                                                     20%                                                        SBS 24,000\u00d7\n\n                                                     10%\n\n                                                      0%\n                                                                   0             0.05              0.1            0.25              0.5\n\n                                                                              Variant allele %\n\n                                                       Data based on internal testing. >4\u00d7 improvement in sequencing efficiency relative to SBS\n                                                                                                                                                  140 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 141](rendered_slides/MQ_Tech_Full-141.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Nanopore Sequencing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 142](rendered_slides/MQ_Tech_Full-142.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Oxford nanopore\n\n\u00bb Sequences single molecules of DNA as travel\n  through pore\n\n\u00bb Can also identify modified bases and\n  sequence RNA directly\n\n\u00bb Reads typically 5-50kb but some much longer\n  (4Mb record) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 143](rendered_slides/MQ_Tech_Full-143.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 144](rendered_slides/MQ_Tech_Full-144.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 145](rendered_slides/MQ_Tech_Full-145.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- ONT Simplex (1D) Library prep -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 146](rendered_slides/MQ_Tech_Full-146.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 147](rendered_slides/MQ_Tech_Full-147.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Flow cell design          Nanopore\n\n\n\n\n\u00bb Application-Specific Integrated\n  Circuits (ASICs), MinION ASIC\n  contains 512 channels\n                                             Synthetic\n                                             membrane\n\n\n\u00bb Each channel is surrounded\n by 4 pores & records only 1\n                                           ASIC\n at a time\n\n\u00bb 512 pores max recorded at the time -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 148](rendered_slides/MQ_Tech_Full-148.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Oxford Nanopore Technologies\n\n\n\n\n      Imagine 5-base words = 1024 current levels -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 149](rendered_slides/MQ_Tech_Full-149.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- ONT: The squiggles\n\n\n\n\n     A G G T G C A T T G C T T G A G T T C T T T C A G T T A C T\n\n\n\n\nSlide courtesy of David Buck. WTCGH -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 150](rendered_slides/MQ_Tech_Full-150.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Assuming 5mer current signals\n\n\n\n\n    From Szalay & Golovchenko, Nat. Biotech., 2015. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 151](rendered_slides/MQ_Tech_Full-151.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- ONT basecalling with machine learning -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 152](rendered_slides/MQ_Tech_Full-152.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- R10. A dual head nanopore to help resolve\n             homopolymers\n\n\n CsgF\n\n\n\n\nCsgG\n\n\n\n\n        From Van der Verren et al., Nature Biotech 2020 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 153](rendered_slides/MQ_Tech_Full-153.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- ONT Sequencing\nperformance graphs -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 154](rendered_slides/MQ_Tech_Full-154.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- ONT accuracy -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 155](rendered_slides/MQ_Tech_Full-155.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- ONT accuracy -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 156](rendered_slides/MQ_Tech_Full-156.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- ONT accuracy -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 157](rendered_slides/MQ_Tech_Full-157.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Direct sequencing of modifications -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 158](rendered_slides/MQ_Tech_Full-158.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- minION\n$1999. Yields upto 48Gb ~$5/Gb.\n          512pores. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 159](rendered_slides/MQ_Tech_Full-159.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Connected minION\nMk Ic\n        Mk Id based on ipad\n        pro -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 160](rendered_slides/MQ_Tech_Full-160.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- GridION. For service sector\n\n\n\n\n$67000 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 161](rendered_slides/MQ_Tech_Full-161.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Flongle flow cell adapter\n    $90 for 2-3Gb -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 162](rendered_slides/MQ_Tech_Full-162.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- PromethION\n            24 or 48 flowcells with 12000 pores\n\n\n\n\n~$7-16/Gb -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 163](rendered_slides/MQ_Tech_Full-163.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- PromethION P2 Solo\n\n\n\n\n~$7-16/Gb -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 164](rendered_slides/MQ_Tech_Full-164.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- ONT unique applications\n\n\u00bb Ultra-long reads (>100kb reads possible)\n\u00bb \u201cRun until\u201d done. W.I.M.P\n\u00bb Selective reads\n\u00bb Mobile sequencing\n\u00bb Direct RNA sequencing\n\u00bb Direct methylation detection over wide range\nof DNA and RNA modifications -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 165](rendered_slides/MQ_Tech_Full-165.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- So. Which to choose?\n\n\u00bb Budget\n\u00bb Technical fit\n\u00bb Applications\n\u00bb Throughput\n\u00bb Peers\n\u00bb Space -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 166](rendered_slides/MQ_Tech_Full-166.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 167](rendered_slides/MQ_Tech_Full-167.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Any Questions ?\n\n   mq1@sanger.ac.uk -->"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "name": "python",
   "version": "3.8.10"
  },
  "celltoolbar": "Slideshow"
 },
 "nbformat": 4,
 "nbformat_minor": 4
}