{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Resource: Oxford_Nanopore_2025_plus_analysis.pdf\n",
    "This notebook is a high-fidelity conversion of the original PDF slide deck."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 1](rendered_slides/ONT_Full-01.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Oxford Nanopore sequencing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 2](rendered_slides/ONT_Full-02.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Oxford Nanopore sequencing\n\nObjective:\n\u2022 To gain a greater depth of understanding of the principles of working\n  with Long Reads and ONT sequencing\n\n\u2022 To understand the ONT offering with regards to making an informed\n  choice for your project -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 3](rendered_slides/ONT_Full-03.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 4](rendered_slides/ONT_Full-04.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 5](rendered_slides/ONT_Full-05.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 6](rendered_slides/ONT_Full-06.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 7](rendered_slides/ONT_Full-07.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 8](rendered_slides/ONT_Full-08.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 9](rendered_slides/ONT_Full-09.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Oxford Nanopore sequencing\n\n\n\n\n       Image c/o Oxford Nanopore, https://www.youtube.com/watch?v=3UHw22hBpAk -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 10](rendered_slides/ONT_Full-10.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Q20+\n\n \u2022 Combination of latest Nanopore(R10.4.1) and Library Construction kit (\u201cKit 14\u201d) will give\n   single strand accuracy of \u201c99.3%\u201d and duplex (where both strands of DNA are\n\n \u2022 basecalled) of \u201c~99.9%\u201d Duplex = methylation information & high quality basecalls from\n\n      single strand \u2013no need for consensus (which can obscure rare variants)\n \u2022 This is currently in early access testing and not fully commercially available Some\n \u2022 applications, e.g. Direct RNA sequencing, may not be available as quickly as others\n\n\n\n N.B. Illumina NovaSeqstats suggest >75% reads at Q30 at 2x250 or >90% reads at Q30 at 2x50bp\n https://emea.illumina.com/systems/sequencing-platforms/novaseq/specifications.html\n                                                                                                Q10 = 90% Q20 = 99% Q30 = 99.9% -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 11](rendered_slides/ONT_Full-11.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- DNA Extraction Methods\n                                         Magnetic\n                                          beads?\n     Kit?                    Nucleus\n                            isolation?\n                                                     Bead\n                Phenol\n              Chloroform?                           beating?\n\n\nYou cannot get long reads from short fragments! -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 12](rendered_slides/ONT_Full-12.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- DNA extraction methods\n\nThe four S\u2019s!\n\nSafety                       What is the best\n                             method for your\nSpecies                       application?\n Suitable (cost,equipment)\nScale up (or size) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 13](rendered_slides/ONT_Full-13.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Assessing your DNA fragments\n\nPurity                   Nanodrop\n\nLength                   Tapestation or FemtoPULSE\n                         or gel\n\nAmount\n                         Qubit -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 14](rendered_slides/ONT_Full-14.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Library construction -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 15](rendered_slides/ONT_Full-15.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- L iga tio n Suited to maximising\n                            Rapid\n                                                                   data yield\n          Suited to fieldwork or ultralongsequencing\nImage c/o Oxford Nanopore -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 16](rendered_slides/ONT_Full-16.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Optional Fragmentation\n\nNot recommended for some applications, e.g. short\nfragments, PCR products\n\nCan be useful when some very long fragments are present\nbut high throughput is priority\n\nOptions include G-TUBE (historically recommended by ONT),\nneedle shearing, Megaruptor -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 17](rendered_slides/ONT_Full-17.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Ligation vs Rapid\n\n                    Ligation   Rapid\n     Yield Time\n     Cost Data\n     quality\n     Robustness -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 18](rendered_slides/ONT_Full-18.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Multiplexing\n\n Make best use of a flow cell capacity by   Component      Sequence\n                                            NB01   AAGAAAGTTGTCGGTGTCTTTGTG\n combining >1 sample together Each          NB02   TCGATTCCGTTTGTAGTCGTCTGT\n                                            NB03   GAGTCTTGTGTCCCAGTTACCAGG\n sample is \u201clabelled\u201d with a known          NB04   TTCGGATTCTATCGTGTTTCCCTA\n DNA sequence                               NB05   CTTGTCCAGGGTTTGTGTAACCTT\n                                            NB06   TTCTCGCAAAGGCAGAAAGTAGTC\n Oxford Nanopore barcodes are longer        NB07   GTGTTACCGTGGGAATGAATCCTT\n                                            NB08   TTCAGGGAACAAACCAAGTTACGT\n than Illumina, at 24 bases\n                                            NB09   AACTAGGCACAGCGAGTCTTGGTT\n                                            NB10   AAGCGTTGAAACCTTTGTCCTCTC\n This allows confidence despite the\n                                            NB11   GTTTCATCTATCGGAGGGAATGGA\n higher per-base error rate, and makes      NB12   CAGGTAGAAAGAAGCAGAATCGGA\n\n\n use of Oxford Nanopore\u2019slong reads -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 19](rendered_slides/ONT_Full-19.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Basecalling\n\nCurrent or recently announced basecallers now allow:\n\n   \u2022Real time methylation information\n   \u2022Demultiplexing of samples\n\n\n   \u2022Accurate sequencing of short fragments (>20bp) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 20](rendered_slides/ONT_Full-20.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Oxford Nanopore Platforms and Products -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 21](rendered_slides/ONT_Full-21.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Oxford Nanopore sequencing\n\n\n\n\n       MinI ON Up               GridI O N Up    Pro methI O N Up to\n\n         to 50Gb                 to 250Gb       220Gb per flow cell\n                                               10.5Tb per instrument\n\n\n\n\n    Image c/o Oxford Nanopore -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 22](rendered_slides/ONT_Full-22.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Images center and right c/o Oxford Nanopore -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 23](rendered_slides/ONT_Full-23.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Why is it so small?\n\n   Other machines need to have cameras and lasers to take pictures of the\n   DNA\n\n   They need to have moving parts to add chemicals to synthesise the\n   DNA\n\n   The MinIONmeasures electrical signals from DNA that is already\n   present -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 24](rendered_slides/ONT_Full-24.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- MinION\n\nDon't forget cost\nof computer!\n\n\n\n\n   Prices c/o Oxford Nanoporestore, taken 27/03/2025 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 25](rendered_slides/ONT_Full-25.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 26](rendered_slides/ONT_Full-26.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 27](rendered_slides/ONT_Full-27.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- GridION\n\n\n\n\n  Image c/o Oxford Nanopore -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 28](rendered_slides/ONT_Full-28.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- GridION\n\n\n\n\n  Image c/o Oxford Nanopore -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 29](rendered_slides/ONT_Full-29.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- PromethION\n                              48 flow cells\n\n                              Each flowcell\n                              originally \u201c50Gb\u201d\n                              now maximum is\n                              290Gb\n\n                              Up to 14Tb of\n                              sequence\n\n                              Rapidly evolving\n\n  Image c/o Oxford Nanopore -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 30](rendered_slides/ONT_Full-30.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- PromethION\n\n\n\n\n  Image c/o Oxford Nanopore -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 31](rendered_slides/ONT_Full-31.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- P2\n\nA \u201cmini\u201d\nPromethION\n\n\nAlso as \u201csolo\u201d model \u2013\nplugs into GridIONor\ncomputer\n\n\n\n\n     Image c/o Oxford Nanopore -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 32](rendered_slides/ONT_Full-32.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 33](rendered_slides/ONT_Full-33.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Flongle\n                             48 Flongle Flow cells\n\n\n\n                                        Adapts MinION to take\n                                        cheap, low throughput flow\n                                        cells\n\n                                        Closed store BUT still\n                                        available\n\n\n\n\n Image c/o Oxford Nanopore -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 34](rendered_slides/ONT_Full-34.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Q Line\n\nSO 9001:2015 certified manufacturing process\n\n12 months guaranteed supported software & consumables\nClearly mapped out upgrades\n\n\nSame price as standard devices & consumables\n\nAllows in house validation of assays for long term use -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 35](rendered_slides/ONT_Full-35.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Where has Oxford Nanopore been used? -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 36](rendered_slides/ONT_Full-36.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- http://www.nature.com/news/pint-sized-dna-sequencer-impresses-first-users-1.17483 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 37](rendered_slides/ONT_Full-37.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 38](rendered_slides/ONT_Full-38.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 39](rendered_slides/ONT_Full-39.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Image c/o NASA, USA -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 40](rendered_slides/ONT_Full-40.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Image from The Guardian Online\nhttps ://www. the g ua rdi a n. c om/s c i e nc e /2016/ f e b/03/f rom- e b ol a - to- zi ka - ti ny- m obi l e - l a b- g i ve s - re a l - ti\nme - dna - da ta - on- outbre a ks -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 41](rendered_slides/ONT_Full-41.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- SARS-CoV-2 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 42](rendered_slides/ONT_Full-42.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Concluding thoughts -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 43](rendered_slides/ONT_Full-43.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Oxford Nanopore sequencing\n\nAdvantages              Disadvantages\nPortable                Can require highly pure DNA\nRapid                   Can require large amountsof input DNA\nLow start up costs      Less supported analysis\nNative DNA sequencing\nLong read\nMultiplexing options\nFleet of sequencers -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 44](rendered_slides/ONT_Full-44.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Floatation\n\nOxford Nanopore Technologies floated on the\nLondon Stock Exchange in 2021\n\nCurrently valued at ~\u00a3979 million ($1.26 billion \u2013for comparison Illumina\n~$12.96 billion) down from about \u00a32.2 billion (Illumina down from $31\nbillion)\n\nWhat does this mean for your science?\n-perhaps greater visibility of how the company is performing\n-otherwise, not much -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 45](rendered_slides/ONT_Full-45.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Additional technologies\n\n\n\n\n                          press.office@sanger.ac.uk -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 46](rendered_slides/ONT_Full-46.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- \u2022All scripts/software = open source\n\n\u2022Slides & links available on your course webpage\n\n\u2022Questions?\n\n   kj6@sanger.ac.uk\n\n   @kim_judge_ -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 47](rendered_slides/ONT_Full-47.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Analysis session -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 48](rendered_slides/ONT_Full-48.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Objective\n\n Check our data\n Assemble bacterial data generated using the latest Oxford Nanopore\n Chemistry (as used this morning)\n Check completeness of assembly\n Investigate assembling other samples\n Each genome can have many assemblies! -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 49](rendered_slides/ONT_Full-49.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Why? -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 50](rendered_slides/ONT_Full-50.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- head\n                  tail\nLet\u2019s find out    ls \u2013l\n\nreads and check   assembly-stats\n\n\nthem out -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 51](rendered_slides/ONT_Full-51.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Depth of Coverage\nand\nTheoretical Depth of Coverage -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 52](rendered_slides/ONT_Full-52.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- 1. Install minimap2 and miniasm\n  Search for the website\n\n  https://github.com/lh3/miniasm\n\n\n  Copy the commands into your terminal\n\n\n\n  git clone https://github.com/lh3/minimap2 && (cd minimap2 && make)\n\n  git clone https://github.com/lh3/miniasm && (cd miniasm && make) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 53](rendered_slides/ONT_Full-53.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- 2. Run the overlap\n\n\nminimap2/minimap2 -x ava-ont </path/to/sample1.fastq> </path/to/sample1.fastq> > <sample1.paf> -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 54](rendered_slides/ONT_Full-54.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- 3. Run the assembly\n\n\n miniasm/miniasm -f </path/to/sample1.fastq> <sample1.paf> > <sample1.gfa>\n\n\n Minimap2 & miniasm will work with compressed (zipped) data files, so you can try running:\n\n\n minimap2/minimap2 -x ava-ont </path/to/reads.fq> </path/to/reads.fq> | gzip -1 >\n <reads.paf.gz>\n\n miniasm/miniasm -f <reads.fq> <reads.paf.gz> > <reads.gfa> -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 55](rendered_slides/ONT_Full-55.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- 4. Assess the assembly\n    Process the assembly to get fasta format\n    awk '$1==\"S\" {print \">\"$2\"\\n\"$3}' sample1.gfa > sample1.fa\n    Then Try:\n    head\n    wc-l\n    tail\n    ls -l\n    assembly-stats -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 56](rendered_slides/ONT_Full-56.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Overview\n minimap2/minimap2 -x ava-ont </path/to/sample1.fastq> </path/to/sample1.fastq> > <sample1.paf>\n\n miniasm/miniasm-f </path/to/sample1.fastq> <sample1.paf> > <sample1.gfa>\n\n awk '$1==\"S\" {print \">\"$2\"\\n\"$3}\u2019 <sample1.gfa> > <sample1.fa>\n\n assembly-stats <sample1.fa> -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 57](rendered_slides/ONT_Full-57.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- 5. Try on other samples -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 58](rendered_slides/ONT_Full-58.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- You could consider submitting part of your finished assembly to BLAST\n\nGoogle NCBI BLAST\n\nClick Nucleotide BLAST\n\nPaste in the box -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 59](rendered_slides/ONT_Full-59.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Round up and final questions -->"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "name": "python",
   "version": "3.8.10"
  },
  "celltoolbar": "Slideshow"
 },
 "nbformat": 4,
 "nbformat_minor": 4
}