{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Resource: NGS_course-Adrian_Cazares.pdf\n",
    "This notebook is a high-fidelity conversion of the original PDF slide deck."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 1](rendered_slides/Assembly_Full-01.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- NGS to trace the\nhistorical spread of\nantimicrobial\nresistance\n Dr. Adrian Cazares\n Sta\ufb00 Scientist - AMR projects lead\n\n                           Nick Thomson Group\n             Parasites and Microbes Programme\n\n                                   March, 2025\n\n        Contact:\nac61@sanger.ac.uk\n    @cazares_adr -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 2](rendered_slides/Assembly_Full-02.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Parasites and Microbes Programme (PAM)\n\n         PAM\u2019s\n     contribution\n     to infectious\n        disease\n       research\n\n\n\n      Thomson\n        team\n      Bacterial\n      Genomics\n         and\n      Evolution\n\nWe use genomics approaches to understand the way pathogens have moved, and continue to move, through populations and across continents\n                       with a view to providing high-resolution insights aimed at tracking and limiting their spread. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 3](rendered_slides/Assembly_Full-03.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Talk outline\n\n\u2794   Mobile Genetic Elements\n    (MGEs) and AMR\n\n\u2794   Evolution of AMR spread\n    during the Yemen cholera\n    outbreak\n\n\u2794   Plasmids evolution\n    throughout the antibiotic\n    epoch\n\n\u2794   Future directions -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 4](rendered_slides/Assembly_Full-04.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- The mobilome\u2019s role in AMR Gene Transfer\n\n     Transfer of DNA between bacterial cells\n                                               \u25cf   MGEs: elements that promote intracellular and\n                                                   intercellular DNA mobility.\n\n\n                                               \u25cf   The concerted activity of both types of MGEs\n                                                   results in Horizontal Gene Transfer.\n\n\n                                               \u25cf   Bacterial pathogens are able to meet the\n                                                   evolutionary challenge of combating\n                                                   antimicrobials, often by acquiring preexisting\n                                                   resistance determinants from the bacterial\n                                                   gene pool.\n\n\n                                               \u25cf   Widespread multidrug resistance is commonly\n                                                   achieved by acquisition of preexisting\n                                                   determinants followed by amplification in\n                                                   response to selection.\n\n\n\n\n                                                                      (Figure) Frost et al. 2005; Hulter et al. 2017 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 5](rendered_slides/Assembly_Full-05.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Plasmids, the main vehicles of AMR spread -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 6](rendered_slides/Assembly_Full-06.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Understanding the emergence\nof multidrug resistance during\nthe Yemen cholera outbreak\n\n\n               Lassalle et al. 2023, Nature Microbiology -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 7](rendered_slides/Assembly_Full-07.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Genomic epidemiology of Vibrio cholerae in Yemen\nPhylogenetic\ndiversity and                   * Cholera is an acute diarrhoeal disease caused by ingesting\nspatio-tempora                  food or water contaminated with the bacterium Vibrio\nl distribution of               cholerae.\nV. cholerae\n7PET-T13                        * Only selected strains have potential to cause epidemics and\nisolates                        pandemics, mainly due to their ability to produce cholera\n(VcH.9) from                    toxin. The current pandemic is associated with the\nYemen.                          emergence of V. cholerae biotype known as 7PET.\n\n                                * V. cholerae reached Yemen in 2016, causing the largest\n                                cholera outbreak in modern history.\n\n                                * Prior genomic surveillance revealed the introduction of the\n                                7PET T13 in Yemen, which was susceptible to all antibiotics.\n\n                                * Ongoing surveillance coupled with genomics and\n                                microbiology characterisation in the region between 2018 and\n                                2019 documented the emergence of plasmid-mediated\n                                multidrug resistance.\n\n\n\n                                                          Lassalle et al. Nature Microbiology. 2023 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 8](rendered_slides/Assembly_Full-08.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- The Yemen MRD plasmid is spreading across the species\n\n Comparison of IncC plasmids pCNRVC190243 (Yemen plasmid) and pYA00120881 (plasmid from V. cholerae\n strains collected in Zimbabwe).\n\n\n\n\n * By looking for the plasmid sequence, we found instances across 7PET and Non-7PET strains, both locally and gobally.\n\n * We also found evidence of movement of the MDR region into the chromosome.\n\n * The MDR region seems to be specific of the Yemen plasmid.\n\n\n                                                                                                 Lassalle et al. Nature Microbiology. 2023 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 9](rendered_slides/Assembly_Full-09.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Take home messages\nExpansion of multidrug resistance in Yemeni 7PET V. cholerae isolates.\n\n\n\n\nGenomic analysis of V. cholerae isolates recovered in Yemen revealed that clinical macrolide use initiated in 2016 to treat cholera preceded a shift in\nV. cholerae genotype from antibiotic susceptibility to multidrug resistance between 2018 and 2019.\n\n\nThis was due to the spread of a newly acquired MDR plasmid conferring resistance (beige) in 7PET epidemic V. cholerae strains (blue).\n\n\nThis MDR plasmid was also carried by non-epidemic V. cholerae (grey) present, indicating that genetic exchange between epidemic and endemic\nstrains may have contributed to the emergence of multidrug resistance.\n\n\n                                                                                                                      Pragasam et al. Nature Microbiology. 2023 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 10](rendered_slides/Assembly_Full-10.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Going back in time to map the\norigins of antimicrobial\nresistance vectors\n\n\n                  Cazares et al. 2025. In revision. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 11](rendered_slides/Assembly_Full-11.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- The historical context of antibiotics discovery\n\n\n          Titanic sinks                                 Great depression                                                                                                  England World\n                          WWI ends (1918)                                                               WWII ends (1945)    Coronation (1952)\n             (1912)                                       starts (1929)                                                                                                 champions (1966)\n\n\n\n\n      1910                         1920                         1930                             1940                      1950                   1960\n\n                                                                                                                Antibiotics Industrialisation & Golden Era\n      Pre-antibiotic Era (PAE)\n                                                                           Fresh air treatment                                                                      Resistance\n                                                   Penicillin discovery\n                                                                              for TB (1936)                                                                         genes\n                                                         (1928)\n\n\n\n                                                                                                                                                 Plasmids confer\n                                                                                                                                                    multidrug\n                                                                                                                                                resistance (1960)\n\n\n\n                                                         The Murray Collection (1917 - 1954)\n                            Prof. Everitt George\n                            Dunne Murray -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 12](rendered_slides/Assembly_Full-12.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Reviving and sequencing pre-antibiotic era pathogens\n\n\n\n\n                                      Cazares et al. In revision. 2025 (Preprint available) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 13](rendered_slides/Assembly_Full-13.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Study overview\n\n\n\n\n                 Cazares et al. In revision. 2025 (Preprint available) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 14](rendered_slides/Assembly_Full-14.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Reconstructed pre-antibiotic era (PAE) plasmids\n\n      PAE plasmids\n      discovered\n\n          PAE plasmids             Murray\n           in GenBank             plasmids\n                                (this study)\n             n=22                  n=765\n\n         35-fold\n         increase\n\n                                         Cazares et al. In revision. 2025 (Preprint available) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 15](rendered_slides/Assembly_Full-15.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Relationship between PAE plasmids and their modern relatives\n\nPAE plasmids were\nhighly diverse  Sequence similarity network\n\n\n                                                 ~10,000 modern\n                                              plasmids linked to PAE\n                                                     relatives\nExamples\nof plasmid\n species\n\n                                              More than 900 plasmid\n                                               \u201cspecies\u201d identi\ufb01ed\n\n\n                      Source: PAE   Modern    Cazares et al. In revision. 2025 (Preprint available) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 16](rendered_slides/Assembly_Full-16.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Plasmids contribution to AMR spread\n\nAMR genes concentrate in a minority\nof plasmids and plasmid species\n                                      Resistance plasmids are\n                                      not randomly distributed\n                                           in the network.\n\n\n                                      Not all plasmids are the\n                                         same. AMR gene\n                                       carriage is con\ufb01ned to\n                                        38% of all plasmids.\n\n                                       Cazares et al. In revision. 2025 (Preprint available) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 17](rendered_slides/Assembly_Full-17.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- PAE plasmids contribution to AMR spread\nPAE plasmids\nevolved into\nresistance\nvectors                             No speci\ufb01c resistance\n                                    genes in PAE plasmids\n\n\n                                        Thousands of\n                                      resistance genes in\n                                       modern relatives\n\n\n                                     Cazares et al. In revision. 2025 (Preprint available) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 18](rendered_slides/Assembly_Full-18.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Evolutionary strategies followed by 244 resistance plasmid\nspecies\n\n       Microevolution                                                                 Modern\n\n\n       PAE         Modern   Resistance\n                            genes\n\n                                                           PAE               Similarity\n\n\n\n\n                                                    PAE\n       Plasmid fusion\n             Unknown\n PAE\n                        Modern                                                       Modern\n PAE\n                                 Resistance\n                                 genes\n                                                                Similarity\n                                              PAE\n\n                                                    Cazares et al. In revision. 2025 (Preprint available) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 19](rendered_slides/Assembly_Full-19.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Take home messages\n\n\n\u2794   Only a minority of PAE plasmid species evolved to carry resistance genes and they\n    share common characteristics and evolutionary strategies.\n\n\u2794   We can focus on these plasmids to enhance genomics surveillance and investigate\n    what makes them good resistance vectors.\n\n\u2794   Humans distorted plasmids identity, their role and the way bacteria face antibiotics.\n    We have promoted cooperation for survival. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 20](rendered_slides/Assembly_Full-20.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Thanks!\n Wendy Figueroa         Daniel Cazares     Zam Iqbal\n                                         Leandro Lima\n\n\n                                                        Nick Thomson\n                                                        Thomson group\n                                                         Onelenna Neo\n     Jake D. Turnbull                                      Arnav Lal\n Hannah Mcgregor Jo                                       PAM core,\n               Dicks                                     samples & IT\n     Sarah Alexander                                        teams\n                                                                        Contact:\n                                                                ac61@sanger.ac.uk\n                                                                    @cazares_adr -->"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "name": "python",
   "version": "3.8.10"
  },
  "celltoolbar": "Slideshow"
 },
 "nbformat": 4,
 "nbformat_minor": 4
}