{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Resource: Current Methodologies for Targeted Sequencing_NGS course_2025.pdf\n",
    "This notebook is a high-fidelity conversion of the original PDF slide deck."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 1](rendered_slides/Targeted_Full-01.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Current Methodologies for\n  Targeted Sequencing\n         Mike Quail\n WGC AC: Next Gen Sequencing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 2](rendered_slides/Targeted_Full-02.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- What is Targeted Sequencing??\n\u201cTargeted resequencing is a variation of re-sequencing\nwhere only a small subset of the genome is sequenced,\n such as the exome, a particular chromosome, a set of\n             genes or a region of interest\u201d -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 3](rendered_slides/Targeted_Full-03.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Overview\n\u2022 Why??\n\u2022 How??\n  \u2013 Hybridisation capture\n  \u2013 Amplicon\n\u2022 Applicability depends upon project\n\u2022 Examples -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 4](rendered_slides/Targeted_Full-04.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Why Targeted Sequencing?\n\u2022 Cheaper \u00a3\u00a3\u00a3\u00a3\u00a3\n\u2022 Diagnostic \u2013 particular diseases\n\u2022 Detect low levels of variation\n\u2022 Genome-wide association studies (GWAS)\n  instead of arrays\n  \u2013 Rare alleles\n\u2022 Genotyping -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 5](rendered_slides/Targeted_Full-05.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- \u00a3100-\u00a3200   \u00a350 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 6](rendered_slides/Targeted_Full-06.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- \u00a3100-\u00a3200\n\n    \u00a350 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 7](rendered_slides/Targeted_Full-07.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Cost\nTarget         Size    Data    Cost\n\n\nHuman Genome   3Gb     90Gb    \u00a3100-\u00a3200\n\n\n\n\nHuman Exome    50Mb    5Gb     \u00a350\n\n\n\n\nCustom panel   <1Mb    0.5Gb   \u00a3100 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 8](rendered_slides/Targeted_Full-08.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Applications -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 9](rendered_slides/Targeted_Full-09.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Diagnostic -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 10](rendered_slides/Targeted_Full-10.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Diagnostic -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 11](rendered_slides/Targeted_Full-11.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 12](rendered_slides/Targeted_Full-12.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Minimal residual disease -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 13](rendered_slides/Targeted_Full-13.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Low level variants -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 14](rendered_slides/Targeted_Full-14.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Genome Wide Association Studies\n            (GWAS)\n\u2022 GWAS based on common SNPs\n  \u2013 contribution of 1000\u2019s of loci to risk for common\n    diseases and to variation in quantitative traits.\n  \u2013 However explain only a fraction of heritable variation -\n    important role for variants not assessed in GWAS.\n  \u2013 In particular, unidentified low-frequency genetic\n    variants may have a major impact on complex\n    phenotypes -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 15](rendered_slides/Targeted_Full-15.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Genotyping\n\u2022 Multiplex enrichment PCR\n  \u2013 PCR primers designed to amplify the targets of\n    interest (known SNPs)\n\u2022 Restriction enzyme (RE) digestion\n  \u2013 Complexity of genome reduced by restriction\n    digestion\n  \u2013 little genome sequence is available or no prior\n    SNP variants have been identified -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 16](rendered_slides/Targeted_Full-16.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- PCR based approaches can be very\nsensitive eg Covid-19 ARTIC method -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 17](rendered_slides/Targeted_Full-17.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 18](rendered_slides/Targeted_Full-18.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Targeted Sequencing? -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 19](rendered_slides/Targeted_Full-19.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- How?\nHybridisation Capture          PCR based\n  (solution/array)\n\n\n      Samples                      Samples\n\n  Library Creation          Generate Amplicons\n\n  Hybrid selection         Add sequencing motifs\n\n    Sequencing                    Sequencing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 20](rendered_slides/Targeted_Full-20.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Hybrid Capture -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 21](rendered_slides/Targeted_Full-21.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- PCR based capture -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 22](rendered_slides/Targeted_Full-22.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Which Technology?\n\n     Method                         Platform        Company          Capital equipment?       Target Size\n  Non-commercial      Multiplex PCR                n/a           N                        <0.02Mb\n                      Molecular Inversion Probes   n/a           N                        0.02Mb-1.5Mb\nCommercial Amplicon   rhAmpSeq                     IDT           N                        Upto 0.2Mb\n                                                                                          upto 5Mb (custom)\n                      Haloplex                     Agilent       N                        32Mb (exome)\n                      Ampliseq                     Ion Torrent   N                        upto 33Mb\n\n\n                      NEB Next Direct              NEB           N                        Upto 0.3Mb\n   Hybridisation      Sure Select                  Agilent       N                        0.001-100Mb\n                      TruSeq                       Illumina      N                        upto 67Mb\n                      EZ-Cap                       Nimblegen     N                        upto 200Mb\n                      X-Gen Lockdown               IDT           N                        upto 60Mb\n                      Target enrichment             Twist        N                         Upto 50Mb\n\n                      Target enrichment             Celemics     N                         Upto 10Mb -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 23](rendered_slides/Targeted_Full-23.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Which Technology?\n                    Capital\n                    equipment           DNA quality\n                    \u00a3\u00a3\u00a3                   FFPE?\n                                                       How many\nGenome?\n                                                       samples?\n\n                           Turnaround\n                              time\n   Size of target                             How much DNA?\n       region -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 24](rendered_slides/Targeted_Full-24.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Size of target\n                             region\n\n\nWhole exome                       Focused Exome\n                         (i.e. disease associated target    Custom Small\n~60Mb (X30)                                                region (X1000)\n                               panel) ~16Mb (X200)\n\n\n\n\n        \u2022 Limited sample multiplexing\n        \u2022 Coverage per target\n        \u2022 Bioinformatics -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 25](rendered_slides/Targeted_Full-25.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Which Technology?\n                                          Capital\n                                          equipment\n                     Custom Small         \u00a3\u00a3\u00a3\n     Turnaround\n                    region (X1000)\n        time\n\n\n                                       How many\n                                       samples?\nDesign\n\nOptimisation\n                                     Kits X16/X96\nCommercial / non-\ncommercial                           1000\u2019s samples???? -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 26](rendered_slides/Targeted_Full-26.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Which Technology?\n\n     Method                         Platform        Company          Capital equipment?       Target Size\n  Non-commercial      Multiplex PCR                n/a           N                        <0.02Mb\n                      Molecular Inversion Probes   n/a           N                        0.02Mb-1.5Mb\nCommercial Amplicon   Ampliseq                     Ion Torrent   N                        upto 33Mb\n                                                                                          upto 5Mb (custom)\n                      Haloplex                     Agilent       N                        32Mb (exome)\n   Hybridisation      Sure Select                  Agilent       N                        0.001-100Mb\n                      TruSeq                       Illumina      N                        upto 67Mb\n                      HyperExome                   Roche         N                        upto 200Mb\n                      X-Gen Lockdown               IDT           N                        upto 60Mb\n                      Target enrichment             Twist        N                         Upto 50Mb\n                      Target enrichment             Celemics     N                         Upto 10Mb -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 27](rendered_slides/Targeted_Full-27.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- @Sanger \u2013 Agilent sureselect or Twist\n\n              Samples\n\n\n\n              Library creation &\n               hybrid selection\n\n              Sequencing & QC -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 28](rendered_slides/Targeted_Full-28.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Hybridisation Capture at Sanger -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 29](rendered_slides/Targeted_Full-29.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Agilent Bravo for hybrid capture\n          automation -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 30](rendered_slides/Targeted_Full-30.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Hybridisation Capture - Examples\n\u2022   Whole Exome\n\u2022   Pan Cancer\n\u2022   Heterogeneous disorders\n\u2022   Hearing loss with many causal genes\n    \u2013 200 to 250 genes may be involved in hereditary HL\n\u2022   Intellectual disabilities\n\u2022   Autism spectrum disorders\n\u2022   Cardiovascular disease\n\u2022   Obesity, diabetes\n\u2022   Minor allele frequencies\n    \u2013 enriched for functional mutations, potentially have medical\n      relevance -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 31](rendered_slides/Targeted_Full-31.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Why Twist\n\u2022 Fast turnaround\n\u2022 Greater uniformity\n\u2022 Cheaper custom panels -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 32](rendered_slides/Targeted_Full-32.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Twist hybrid capture for methylation\n\n\n\n\nTwist has probes for both strands\nTwist can produce more complex probe pools to cover all sequence combinations -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 33](rendered_slides/Targeted_Full-33.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Method                         Platform        Company          Capital equipment?       Target Size\n  Non-commercial      Multiplex PCR                n/a           N                        <0.02Mb\n                      Molecular Inversion Probes   n/a           N                        0.02Mb-1.5Mb\nCommercial Amplicon   rhAmpSeq                     IDT           N                        Upto 0.2Mb\n                                                                                          upto 5Mb (custom)\n                      Haloplex                     Agilent       N                        32Mb (exome)\n                      Ampliseq                     Ion Torrent   N                        upto 33Mb\n\n\n                      NEB Next Direct              NEB           N                        Upto 0.3Mb\n   Hybridisation      Sure Select                  Agilent       N                        0.001-100Mb\n                      TruSeq                       Illumina      N                        upto 67Mb\n                      EZ-Cap                       Nimblegen     N                        upto 200Mb\n                      X-Gen Lockdown               IDT           N                        upto 60Mb\n                      Target enrichment             Twist        N                         Upto 50Mb\n\n                      Target enrichment             Celemics     N                         Upto 10Mb -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 34](rendered_slides/Targeted_Full-34.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Bias\n\n\n3 logs of variation\n\n                      >1000x depth -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 35](rendered_slides/Targeted_Full-35.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Agilent Haloplex\nHigh sensitivity. Adds UMIs. Good for deep\n    sequencing of small capture areas -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 36](rendered_slides/Targeted_Full-36.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Commercial Amplicon\n\u2022 Haloplex (16/48 samples)\n\n\n\n\n       No capital equipment required -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 37](rendered_slides/Targeted_Full-37.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Ampliseq (Ion Torrent) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 38](rendered_slides/Targeted_Full-38.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 39](rendered_slides/Targeted_Full-39.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 40](rendered_slides/Targeted_Full-40.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 41](rendered_slides/Targeted_Full-41.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 42](rendered_slides/Targeted_Full-42.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 43](rendered_slides/Targeted_Full-43.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 44](rendered_slides/Targeted_Full-44.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Truseq Illumina Library and Primer Schema\n\n\n\n\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026..\u2026\u2026\u2026\u2026--i5 index read-\uf0e0\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026..\u2026\u2026\u2026--Read 01-\uf0e0\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026--i7 index read-\uf0e0\n \u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026ACACTCTTTCCCTACACGACGCTCTTCCGATCT \uf0e0\u2026\u2026\u2026.\u2026 GATCGGAAGAGCACACGTCTGAACTCCAGTCAC \uf0e0\n5\u2019\u2026AATGATACGGCGACCACCGAGATCTACAC[i5]ACACTCTTTCCCTACACGACGCTCTTCCGATCTxxxxxxxxAGATCGGAAGAGCACACGTCTGAACTCCAGTCAC[i7]ATCTCGTATGCCGTCTTCTGCTTG\u20263\u2019\n3\u2019\u2026TTACTATGCCGCTGGTGGCTCTAGATGTG[i5]TGTGAGAAAGGGATGTGCTGCGAGAAGGCTAGAxxxxxxxxTCTAGCCTTCTCGTGTGCAGACTTGAGGTCAGTG[i7]TAGAGCATACGGCAGAAGACGAAC\u20265\u2019\n\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026..<-TCTAGCCTTCTCGTCTGCAGACTTGAGGTCAGTG\u2026\u20265\u2019\n\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026\u2026<-Illumina PE Read 2 Sequencing Primer\n\u2026\u2026\u2026\uf0df------------P5------------\uf0e0[i5]\uf0df------------SBS-------------\uf0e0<insert>\uf0df------------------SBS-----------------\uf0e0[i7]\uf0df----------P7--------\uf0e0\n\n                                    <--TGTGAGAAAGGGATGTGCTGCGAGAAGGCTAGA -5\u2018   i5 index primer -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 45](rendered_slides/Targeted_Full-45.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Truseq style PCR indexing primers\n\nThe IDT and other available UDI barcoding primers will have sequence:\nP7\nCAAGCAGAAGACGGCATACGAGAT[barcode]GTGACTGGAGTTCAGACGTGTGCTCTTCCGATC*T\n\nP5 indexing oligo\nAATGATACGGCGACCACCGAGATCTACAC[barcode]ACACTCTTTCCCTACACGACGCTCTTCCGATC*T\n\n\nYour PCR primers will need Illumina flanking sequences at 5\u2019 end compatible with these sequences so should have general design\n\nForward primer would have design\n5\u2019 ACACTCTTTCCCTACACGACGCTCTTCCGATCT gene specific sequence 3\u2019\n\nReverse primer would have design\n5\u2019 GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT gene specific sequence 3\u2019\nUsing these sequences means that custom sequencing primers are not required. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 46](rendered_slides/Targeted_Full-46.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 47](rendered_slides/Targeted_Full-47.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Multiplex PCR design\n\u2022 Design difficulties\n      \u2013 Specificity\n      \u2013 Cross primer interactions (dimers/non\n        specific amplification)\n\n\n\u2022 Need good design software!!\n  Mprimer\n      \u2013 Thorough design algorithms\n      \u2013 Primer 3 input\n            \u2022 Input target and surrounding sequence\n            \u2022 Constrain target sizes to 190-250bp\n            \u2022 Annealing temperature\n\n\nZhiyong Shen et. al. MPprimer: a program for reliable multiplex PCR primer design. BMC Bioinformatics,\n2010, 11:143. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 48](rendered_slides/Targeted_Full-48.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 49](rendered_slides/Targeted_Full-49.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 50](rendered_slides/Targeted_Full-50.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Multiplex PCR Targeted Sequencing \u2013\n               Suited to the Miseq\n\n Multiplex PCR\nreactions in each\n       well\n\n\n\n\n                    Barcoding\n                                MiSeq\u00ae\n\n\n\n\n                       PCR\n\n 384 DNA samples                         Sequencing   Data analysis\n                                           (MiSeq)      (MIDAS) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 51](rendered_slides/Targeted_Full-51.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Study Cohort of healthy individuals\n\n\n                                                                                         15 HOTSPOTS\n                900\n                                                                                        AML gene   Codon\n                         n=4219\nNo of samples\n\n\n\n\n                800                                                   WTCCC             DNMT3A      R882\n                                                                                          JAK2      V617\n                700                                                   UKHLS              NPM1       L287\n                                                                                         SRSF2      P95\n                600\n                                                                                         SF3B1      K666\n                500                                                                      SF3B1      K700\n                                                                                          IDH1      R132\n                400                                                                       IDH2      R140\n                                                                                          IDH2      R172\n                300\n                                                                                         KRAS       G12\n                200                                                                      NRAS       G12\n                                                                                         NRAS       Q61\n                100                                                                        KIT     D816\n                                                                                          FLT3     D835\n                 0\n                                                                                          FLT3     N676\n                      17-29   30-39   40-49   50-59   60-69   70-79     80-89   90-98\n\n                                                                  Age (Years) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 52](rendered_slides/Targeted_Full-52.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Variant Allele Fraction (VAF) quantitation\n              Reproducibility\n\n\n  n=384 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 53](rendered_slides/Targeted_Full-53.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Prevalence and patterns of Clonal Haemopoiesis in\n                     4219 healthy individuals\n\n\n\n\n                                      Spliceosome\n                                      mutations in\n                                      advancing age\n\n\n\n\nMcKerrell et al. Cell Reports 2015. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 54](rendered_slides/Targeted_Full-54.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 55](rendered_slides/Targeted_Full-55.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 56](rendered_slides/Targeted_Full-56.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 57](rendered_slides/Targeted_Full-57.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Targeted sequencing by:\n-PCR\n-Hybrid capture\n-Adaptive sampling\n-cas9 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 58](rendered_slides/Targeted_Full-58.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Adaptive sampling\nSamples not of interest rejected electronically -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 59](rendered_slides/Targeted_Full-59.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Cas9 enrichment\nhttps://www.nature.com/articles/s41587-020-0407-5 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 60](rendered_slides/Targeted_Full-60.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Targeted sequencing by:\n-PCR\n-Hybrid capture\n-cas9 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 61](rendered_slides/Targeted_Full-61.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 62](rendered_slides/Targeted_Full-62.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "name": "python",
   "version": "3.8.10"
  },
  "celltoolbar": "Slideshow"
 },
 "nbformat": 4,
 "nbformat_minor": 4
}