{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Resource: Libraries 2025.pdf\n",
    "This notebook is a high-fidelity conversion of the original PDF slide deck."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 1](rendered_slides/Libs_Full-001.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- NGS Library\n       Construction\n\n         Michael Quail\n        mq1@sanger.ac.uk\nWTAC -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 2](rendered_slides/Libs_Full-002.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Talk Outline\n\n\n\u00bb What is a NGS sequencing library?\n\u00bb How are they made? Options and considerations.\n\u00bb Problems -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 3](rendered_slides/Libs_Full-003.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Sequencers -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 4](rendered_slides/Libs_Full-004.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 5](rendered_slides/Libs_Full-005.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- 1   tcctggcatc agttactgtg ttgactcact cagtgttggg atcactcact ttccccctac\n61 aggactcaga tctgggaggc aattaccttc ggagaaaaac gaataggaaa aactgaagtg\n121 ttactttttt taaagctgct gaagtttgtt ggtttctcat tgtttttaag cctactggag\n181 caataaagtt tgaagaactt ttaccaggtt ttttttatcg ctgccttgat atacactttt\n241 caaaatgctt tggtgggaag aagtagagga ctgttatgaa agagaagatg ttcaaaagaa\n301 aacattcaca aaatgggtaa atgcacaatt ttctaagttt gggaagcagc atattgagaa\n361 cctcttcagt gacctacagg atgggaggcg cctcctagac ctcctcgaag gcctgacagg\n421 gcaaaaactg ccaaaagaaa aaggatccac aagagttcat gccctgaaca atgtcaacaa\n481 ggcactgcgg gttttgcaga acaataatgt tgatttagtg aatattggaa gtactgacat\n541 cgtagatgga aatcataaac tgactcttgg tttgatttgg aatataatcc tccactggca -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 6](rendered_slides/Libs_Full-006.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Sequencing workflow\n\n\nLibrary\n Library   Sequencing    Analysis\n Prep       Sequencing    Analysis\n  Prep -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 7](rendered_slides/Libs_Full-007.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Library\n Library   Sequencing    Analysis\n Prep       Sequencing    Analysis\n  Prep -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 8](rendered_slides/Libs_Full-008.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Library\n Library   Sequencing    Analysis\n Prep       Sequencing    Analysis\n  Prep -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 9](rendered_slides/Libs_Full-009.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Library\n Library   Sequencing    Analysis\n Prep       Sequencing    Analysis\n  Prep -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 10](rendered_slides/Libs_Full-010.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- \u00bb Original DNA\n\n\n\n\n\u2022 Library -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 11](rendered_slides/Libs_Full-011.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- \u2022 Library -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 12](rendered_slides/Libs_Full-012.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- \u2022 Illumina Library\n\n\n\n\n  P5                   SP 1\n\n\n\n                                     SP2                   P7\n  Flowcell annealing   Primer site\n                                     Primer site   Flowcell annealing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 13](rendered_slides/Libs_Full-013.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Illumina Library Insert sizes\n          Exome, targeted, RNAseq, ChiP, ATAC\n     P5               SP 1          100-200bp\n\n\n                                                 SP2                         P7\n                      Primer site\n                                                 Primer site\n\nReads need to be just long enough to map and ideally not to overlap exon boundaries\n\n\n\n\n    Whole Genome\n     P5               SP 1           400-600bp                                    P7\n\n\n                                                               SP2\n                      Primer site\n                                                               Primer site\n\n    Reads need to be just long to span common repeat elements eg AluI\n    with unique sequence on either side so can map -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 14](rendered_slides/Libs_Full-014.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Library Prep Approaches\n\n1. Classical adapter ligation method\n\n2. Transposon mediated\n\n3. Inclusion of adapter sequences during\n   PCR -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 15](rendered_slides/Libs_Full-015.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Illumina Paired End\n5\u2019\n                                Library Prep\n                        P\n\n\n          T                 5\u2019                         A\n          P\n                    +       A                          5\u2019\n                                                            P\n\n\n\n3\u2019                                       T4 DNA Ligase\n                                                                                     R1 sequencing primer\n\n              5\u2019                                                      3\u2019\n\n\n                                 T                       A\n                                                         T\n                             A\n\n                                                                                     Reverse compliment of R2 primer\n                                                                      5\u2019\n              3\u2019\n\n                   P5                    Hybridize primers\n\n\n              5\u2019                                                      3\u2019   P7\n\n\n                             T                           A\n                                                         T\n                             A\n\n\n                                                                      5\u2019\n              3\u2019\n                                         Limited PCR\n\n\n\n\n     5\u2019                              T                      A                   3\u2019\n     3\u2019                              A                      T                   5\u2019\n\n                                         Make clusters and sequence -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 16](rendered_slides/Libs_Full-016.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Nextera -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 17](rendered_slides/Libs_Full-017.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Amplicon sequencing\n add adapter sequences to 5\u2019 end of primers -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 18](rendered_slides/Libs_Full-018.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Illumina Library Prep -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 19](rendered_slides/Libs_Full-019.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- How do you go about making a\ngood library? -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 20](rendered_slides/Libs_Full-020.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- QC -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 21](rendered_slides/Libs_Full-021.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Input DNA for NGS\n\n\u2022 Properly quantified. Flourimetric methods preferred\n\n\u2022 DNA from appropriate source\n\n\u2022 DNA >40kb required for long read applications\n      Pure DNA 260:280 close to 1.8\n      Nanodrop reading should be approx. 2x qubit\n\n\u2022 RNA RIN>8 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 22](rendered_slides/Libs_Full-022.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Initial Characterisation/QC\n\n\u2022 Spectrophotometry (inc nanodrop) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 23](rendered_slides/Libs_Full-023.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Initial Characterisation/QC\n\n\u2022 Spectrophotometry (inc nanodrop) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 24](rendered_slides/Libs_Full-024.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Initial Characterisation/QC\n\u2022 Gel     M       M\n\n\n\n\n\u2022 Qubit -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 25](rendered_slides/Libs_Full-025.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Advanced Analytical\nFragment Analyzer -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 26](rendered_slides/Libs_Full-026.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Advanced Analytical Femto Pulse -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 27](rendered_slides/Libs_Full-027.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Agilent Tapestation -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 28](rendered_slides/Libs_Full-028.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Agilent Tapestation for sequencing\n             library QC\nOffer range of kits for different input and size ranges -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 29](rendered_slides/Libs_Full-029.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Generic library prep workflow 2006\n\nDNA\n\nShear\n   cleanup\nEnd-repair\n   cleanup\nA-Tail\n   cleanup\nLigate to adapter\n    cleanup\nSize select\n\nPCR\n    cleanup\nQuantitate + Characterise -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 30](rendered_slides/Libs_Full-030.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Streamlining generic library prep workflows\n\nDNA                          DNA\n\nShear                        Shear\n   cleanup                      cleanup\nEnd-repair                   End-repair and A-tail\n   cleanup\nA-Tail                       Ligate to adapter\n   cleanup                       cleanup\nLigate to adapter            Size select\n    cleanup\nSize select                  PCR\n                                 cleanup\nPCR                          Quantitate + Characterise\n    cleanup\nQuantitate + Characterise -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 31](rendered_slides/Libs_Full-031.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Shearing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 32](rendered_slides/Libs_Full-032.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- DNA Shearing by Acoustics\n\n\n  AFA tube\n                         Focused acoustics\n\n\n\n\n                                             Reproducible\n       Programmable shearing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 33](rendered_slides/Libs_Full-033.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Shearing\n\n                           Other options\n\n     \u2022 Matrical sonicman (96 well sonication)\n       www.matrical.com\n     \u2022 Bioruptor www.diagenode.com              physical\n     \u2022 Episonic. www.epigentek.com\n     \u2022 Nebulisation (single sample)\n     \u2022 Fragmentase (NEB)\n     \u2022 Nextera (Epicentre)\n                                                enzymatic -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 34](rendered_slides/Libs_Full-034.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Shearing\n\n                           Other options\n\n     \u2022 Matrical sonicman (96 well sonication)\n       www.matrical.com\n     \u2022 Bioruptor www.diagenode.com              physical\n     \u2022 Episonic. www.epigentek.com\n     \u2022 Nebulisation (single sample)\n     \u2022 Fragmentase (NEB)\n     \u2022 Nextera (Epicentre)\n                                                enzymatic -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 35](rendered_slides/Libs_Full-035.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Shearing Problems\n\n - Enzymatic shearing can result in variation in fragment size\n between samples\n    -Can be dependent on amount, GC content or presence of contaminants\n - Sonication based methods can generate heat leading to AT\n   dropout\n - Fragments smaller than the read length can give rise to low ratio\n   of mapped reads.\n - Smaller fragments cluster more efficiently so if multiplexing\n   libraries they should be sheared to similar fragment size ranges\n   else some will be represented more than others in final sequence\n   output. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 36](rendered_slides/Libs_Full-036.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- After shearing\n\n    3\u2019           3\u2019\n\n    5\u2019           5\u2019\n\n\n\n    3\u2019           3\u2019\n\n    5\u2019           5\u2019 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 37](rendered_slides/Libs_Full-037.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- End-Repair\n\np\n            3\u2019\n\n            5\u2019\n        p -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 38](rendered_slides/Libs_Full-038.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Tailing\n\np\n              A 3\u2019\n\nA                 5\u2019\n              p -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 39](rendered_slides/Libs_Full-039.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Ligation of Illumina Adapters\n\n               p\n                       A 3\u2019\n\n               A           5\u2019\n                       p\n5\u2019                              p    3\u2019\n           T\n\n                                T\n3\u2019         p                         5\u2019 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 40](rendered_slides/Libs_Full-040.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- After Ligation\n\n\n5\u2019                    3\u2019\n     T          A\n\n     A           T\n3\u2019                    5\u2019\n\n\n\n                 5\u2019\n                       T\n\n\n                 3\u2019        p -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 41](rendered_slides/Libs_Full-041.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- SPRI: Ampure\n    \u00bb    http://www.beckmangenomics.com/products.html\n\n\nDNA\n\nShear\n\nEnd-repair\n\nA-Tail\n\nLigate to adapter\n\nPCR\n                            \u00bb   Advantages:\nQuantitate + Characterise       Easy to automate, some size selection properties\n\n                            \u00bb   Disadvantages:\n                                A bit fiddly to do manually -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 42](rendered_slides/Libs_Full-042.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Amplification\n     5\u2019                              3\u2019\n          T                A\n\n          A                T\n     3\u2019         4 - 10 cycles, PCR   5\u2019\n\n5\u2019\n5\u2019   5\u2019    T                A         3\u2019   3\u2019\n3\u2019         A                T              5\u2019\n\n\n\n5\u2019         T                A              3\u2019\n3\u2019   3\u2019    A                T         5\u2019   5\u2019\n                                           5\u2019 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 43](rendered_slides/Libs_Full-043.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Multiplexing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 44](rendered_slides/Libs_Full-044.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Multiplexing\n      Tag sequences\n      Tag 1    CGTGAT\n      Tag 2    ACATCG\n      Tag 3    GCCTAA\n      Tag 4    TGGTCA\n      Tag 5    CACTGT\n      Tag 6    ATTGGC\n      Tag 7    GATCTG\n      Tag 8    TCAAGT\n      Tag 9    CTGATC\n      Tag 10   AAGCTA\n      Tag 11   GTAGCC\n      Tag 12   TACAAG -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 45](rendered_slides/Libs_Full-045.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Multiplexing on Illumina\n\n\n\n\n@IL22_1561:1:1:1771:914\nTTTTTTCAATACAATTTCATCTCTAATTTCAATTCAGCCTAA\n+\n>>>>7>>>>>>>>>>>>>>>>>>>:>5>>>96>+>>7;>>>>\n@IL22_1561:1:1:1222:1105\nATGATATCATCTACACGGTTTAAAAATTCTGGACGGCTGATC\n+\n>>;>>>>>>>;>>>>>55>>>>>>>>>>>>+2>>8>>>>>57\n@IL22_1561:1:1:1438:709\n                                              TAG      Strain\nACACCGATGACAATAATTGTTCCAATATCTGTAACAAAGCTA\n                                              CGTGAT   HGSA942\n+\n                                              ACATCG   3HK\n<<<<<2<<2<<<<<<<<<)'<<<<<<<<<<+<;;7<)<<2<<\n                                              GCCTAA   CHI59\n@IL22_1561:1:1:1671:1462\n                                              TGGTCA   CHI61\nTGCAAGAACATTAGACAACGTATCTTCAATCGTTTATACAAG\n                                              CACTGT   BK2491\n+\n                                              ATTGGC   TUR9\n>>7>:>:>>7>>7/:>>>7/.7>>>>>57>>0>>>>>>7>>>\n                                              GATCTG   HU25\n@IL22_1561:1:1:1168:891\n                                              TCAAGT   HU109\nAATGGAAATCAAACTATAACTTCAACACTAAATGAGAAGCTA\n                                              CTGATC   HSJ216\n+\n                                              AAGCTA   FFP103\n>>>:7>>>>>>>>>>>>>>>>>>>>>>>>>>>>0>>>>>>>>\n                                              GTAGCC   ICP5062\n@IL22_1561:1:1:1097:1281\n                                              TACAAG   GRE4\nTAAAAGTATAACTTTCCATCACCAAAGTCAGAATTTACATCG\n+\n>>>>>>>>>>>>>>>>>>>>>>>>8>8>>>>>>>>>>>>>>> -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 46](rendered_slides/Libs_Full-046.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Multiplexing Options\n\n\n\n                       Truseq stubby Adapters\n\n\n                       Stubby Adapters available from IDT\nFull length Adapters\n                       Indexing primers from:\nAvailable from:        Illumina\n                       IDT\nIllumina               NEB\nIDT                    Roche\nNEB                    Quantabio\nRoche                  Tecan\nQuantabio              Perkin Elmer\nTecan                  Eurofins\nPerkin Elmer           etc\netc -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 47](rendered_slides/Libs_Full-047.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- NGS Library Prep Kits\n     -Enzymatic or physical shearing options\n     -With or without adapters/primers\n     -PCR or PCRfree -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 48](rendered_slides/Libs_Full-048.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- NGS library kit manufacturers\n\n\u00bb NEB                \u00bb Agilent\n\u00bb Illumina           \u00bb Tecan\n\u00bb IDT                \u00bb Thermo\n\u00bb Twist bioscience   \u00bb Takara\n\u00bb Roche/Kapa         \u00bb Watchmaker\n\u00bb Qiagen             \u00bb Meridian\n\u00bb Agilent            \u00bb Quantabio\n\u00bb Tecan              \u00bb Thermo -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 49](rendered_slides/Libs_Full-049.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- NEB UltraExpress FS DNA\n   library kit E3340L -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 50](rendered_slides/Libs_Full-050.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Size Selection -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 51](rendered_slides/Libs_Full-051.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Size Selection\n\n\u00bb Gel\n\u00bb Pippin Prep\n\u00bb Ampure -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 52](rendered_slides/Libs_Full-052.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- SPRI: Ampure\n    \u00bb    http://www.beckmangenomics.com/products.html\n\n\nDNA\n\nShear\n\nEnd-repair\n\nA-Tail\n\nLigate to adapter\n\nPCR\n                            \u00bb   Advantages:\nQuantitate + Characterise       Easy to automate, some size selection properties\n\n                            \u00bb   Disadvantages:\n                                A bit fiddly to do manually -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 53](rendered_slides/Libs_Full-053.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- SPRI purification -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 54](rendered_slides/Libs_Full-054.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Working with SPRI Beads\n\n\u00bb   SPRI (Solid Phase Reversible Immobilization) Beads\n\u00bb   Paramagnetic beads made of polystyrene surrounded by a layer of\n    magnetite, which is coated with carboxyl molecules\n\u00bb   Reversibly bind DNA in the presence of the \u201ccrowding agent\u201d polyethylene\n    glycol (PEG) and salt (20% PEG, 2.5M NaCl is the magic mix).\n\u00bb   PEG causes the negatively-charged DNA to bind with the carboxyl groups on\n    the bead surface.\n\u00bb   DNA fragment size affects the total charge per molecule with larger DNAs\n    having larger charges; this promotes their electrostatic interaction with the\n    beads and displaces smaller DNA fragments.\n\u00bb   The size of fragments eluted from the beads is determined by the\n    concentration of PEG, which is determined by the mix of SPRI:DNA at a\n    given ratio.\n\u00bb   The lower the ratio of SPRI:DNA the higher the molecular weight\n    of the fragments bound to the beads. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 55](rendered_slides/Libs_Full-055.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Double sided SPRI\n\n\n\n\u2022 By implementing a combination of good shearing with SPRI and \u201creverse\u201d SPRI, one can select a size range:\n\n\n\n\nSource: Enseqlopedia from Broad Institute and Illumina -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 56](rendered_slides/Libs_Full-056.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Phased Cleanup: removal of adaptors\n\n\n                                                                      Discard                        Add Bead\n      Add beads                                                                        Add\n                                                                        SN                         Reconstitution\n        (0.7X)                                                                       0.1X TE\n                                                                     (adaptors)                     Buffer (0.8X)\n\n\n\n\n    DNA sample          Library binding   Separate beads              Keep Library          Elute DNA            Rebind DNA\n        after           to SPRI beads        from SN                 on SPRI beads         from beads           to SPRI beads\n   PCR enrichment            5 min             5 min                                                                5 min\n\n\n\n          Remove       2x Wash                                                                    Transfer SN\n                                                             Add                                  to new tube\n            SN           with                              0.1X TE                                  (Library)\n         (adaptors)   80% EtOH\n\n\n\n\nSeparate beads              Wash DNA         Air dry             Elute DNA            Separate beads            Cleaned up\n   from SN                  2x 30 sec      bead pellet           In 0.1X TE              from SN                  Library\n     5 min                                   5 min                  2 min                  5 min -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 57](rendered_slides/Libs_Full-057.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Phased Bead Cleanup for UltraExpress RNA workflow\n\n        Add 70 \u00b5l (0.7X)                                                     Remove tubes from magnet\n     resuspended beads to\n    the 100 \u00b5l PCR reaction\n\n\n\n\n\u2022   Pipet mix or quick                              Without disturbing the\n                              Incubate on magnet\n    vortex/centrifuge                              beads carefully remove\n                                 for 5 minutes\n\u2022   Room Temp 5 minutes                            and discard supernatant\n                                                                                 Add 100 \u00b5l of 0.1X\n                                                                                 TE and pipet to mix -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 58](rendered_slides/Libs_Full-058.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Phased Bead Cleanup for UltraExpress RNA workflow\n\n\n\n\nAdd 80 \u00b5l (0.8X) NEBNext Bead Reconstitution Buffer to the 100 \u00b5l resuspended beads\n\n\n\n\n    Pipet mix and            Incubate on magnet         Discard                       200 \u00b5l (80% ETOH)\nincubate at RT 5 min                5 min             supernatant                       wash 2X for 30\n                                                                                           seconds -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 59](rendered_slides/Libs_Full-059.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Phased Bead Cleanup for UltraExpress RNA workflow\n\n\n  Be sure to remove all                  Note: Do not allow beads to over dry or crack\nresidual 80% ETOH and\n  incubate up to 5 min\n\n\n\n\nWhen beads are ready\nelute with 0.1X TE and\n    incubate at RT\n\n\n\n                              Separate                        Transfer eluate\n                             on magnet                         to a new tube -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 60](rendered_slides/Libs_Full-060.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- NEB phased cleanup from\nNEBNext UltraExpress\u2122 FS DNA Library\nPrep Kit\n                     Add 0.8x\n 0.7x                NEB bead\n            100ul    suspension\n            0.1xTE   buffer -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 61](rendered_slides/Libs_Full-061.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Ampure Size Selection -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 62](rendered_slides/Libs_Full-062.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Analysis / Library QC -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 63](rendered_slides/Libs_Full-063.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Final Analysis -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 64](rendered_slides/Libs_Full-064.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Problems -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 65](rendered_slides/Libs_Full-065.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- What can possibly go wrong!! -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 66](rendered_slides/Libs_Full-066.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Adapter Dimer -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 67](rendered_slides/Libs_Full-067.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 68](rendered_slides/Libs_Full-068.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Small Insert Contamination -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 69](rendered_slides/Libs_Full-069.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Small Insert Contamination -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 70](rendered_slides/Libs_Full-070.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Larger fragments??\n\n\n\n\n                 17/9/12 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 71](rendered_slides/Libs_Full-071.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- When primers deplete\nP5 and P7 can anneal but internal\n fragments don\u2019t match up so are\n        single stranded\n        P5                                                              P7\n\n\n\n\nProblem: DNA binding dyes used in quantitation eg Qubit and in tapestation/bioanalyzer\nbind ds DNA so don\u2019t properly quantify\nNeed to qPCR to properly quantify -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 72](rendered_slides/Libs_Full-072.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Bias -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 73](rendered_slides/Libs_Full-073.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- P. falciparum with original Illumina sample prep\n\n\n\n\nGC content\n\ncoverage -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 74](rendered_slides/Libs_Full-074.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Required Sequence Coverage -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 75](rendered_slides/Libs_Full-075.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Required Sequence Coverage\n\n\n\n\n30x sequencing depth required for efficient calling of heterozygous variants\nAim for 95% > 15x so you have sufficient coverage over most\npositions to call variants -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 76](rendered_slides/Libs_Full-076.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Required Sequence Coverage\n\n\n\n\n                    15x\n\n\nIf we get bias the amount of regions with low coverage will increase and we\nwont be able to call variants so we then have a choice of whether to fail\nthose samples or sequence again -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 77](rendered_slides/Libs_Full-077.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Contributors to bias\n\n\u00bb Heat\n\n\u00bb Shearing method\n\n\u00bb PCR\n\n\u00bb Target enrichment -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 78](rendered_slides/Libs_Full-078.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Contributors to bias\n\n\u00bb Heat\n  \u00bb Heat steps >65C can irreversibly denature AT rich regions\n\n\u00bb Shearing method\n\n\u00bb PCR\n\n\u00bb Target enrichment -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 79](rendered_slides/Libs_Full-079.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Contributors to bias\n\n\u00bb Heat\n  \u00bb Heat steps >65C can irreversibly denature AT rich regions\n\n\u00bb Shearing method\n\n\u00bb PCR\n\n\u00bb Target enrichment -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 80](rendered_slides/Libs_Full-080.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Contributors to bias\n\n\u00bb Heat\n  \u00bb Heat steps >65C can irreversibly denature AT rich regions\n\n\u00bb Shearing method\n\n\u00bb PCR\n  \u00bb Need right Enzyme and conditions\n\n\u00bb Target enrichment -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 81](rendered_slides/Libs_Full-081.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Coverage across a section of B. pertussis genome\n\n\n\n        %GC\n\n\n\n\nCoverage depth\n\n\n\n\n                                          Kapa HiFi\n                                          Phusion\n                                          noPCR\n\n                 See https://www.nature.com/articles/nmeth.1814\n                 and https://doi.org/10.1099/mgen.0.001228 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 82](rendered_slides/Libs_Full-082.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Recommended PCR enzymes\n    for NGS library prep\n\u00bb Quantabio repliQa\n\u00bb Watchmaker Equinox\n\u00bb Takara Premier plus\n\nFor Long Read\n\u00bb Quantabio repliQa\n\u00bb Merck KOD Extreme -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 83](rendered_slides/Libs_Full-083.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Contributors to bias\n\n\u00bb Heat\n  \u00bb Heat steps >65C can irreversibly denature AT rich regions\n\n\u00bb Shearing method\n\n\u00bb PCR\n  \u00bb Need right Enzyme and conditions\n\n\u00bb Target enrichment\n  \u00bb 100x coverage required for hybrid capture target enrichment\n  \u00bb 1000x coverage required for PCR based targeted sequencing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 84](rendered_slides/Libs_Full-084.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Sample Representation Bias -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 85](rendered_slides/Libs_Full-085.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Sample representation\n\n\n\n\n          8.7%CV -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 86](rendered_slides/Libs_Full-086.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Sample representation\n\n\n\n\n          8.7%CV -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 87](rendered_slides/Libs_Full-087.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Sample representation\n\n\n\n\n          13%CV -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 88](rendered_slides/Libs_Full-088.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Measuring Sequencer Size Bias Using REcount: A Novel\n            Method for Highly Accurate\n     Illumina Sequencing-Based Quantification\n\n\n\n\n                                          MiSeq\n                                          NextSeq\n                                          Novaseq\n                                          iSeq -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 89](rendered_slides/Libs_Full-089.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- The secret to a low sample\n      representation CV\n\u00bb Standardise as much as possible\n\n  \u00bb Automation\n  \u00bb Reliable quantification step\n  \u00bb Standardise DNA input amounts\n  \u00bb Accurate pipetting\n  \u00bb Control fragment sizes -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 90](rendered_slides/Libs_Full-090.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Efficiency -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 91](rendered_slides/Libs_Full-091.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Library Prep Efficiency -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 92](rendered_slides/Libs_Full-092.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Illumina library efficiency -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 93](rendered_slides/Libs_Full-093.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Illumina library efficiency -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 94](rendered_slides/Libs_Full-094.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 95](rendered_slides/Libs_Full-095.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Library quantification prior to\n         sequencing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 96](rendered_slides/Libs_Full-096.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Library quantification prior to\n         sequencing\n  Options:\n  qPCR\n  Agilent bioanalyser, Caliper\n  Labchip GX\n  Qubit -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 97](rendered_slides/Libs_Full-097.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- qPCR quantification\n\n\n\n                                                     Libraries of\n     200pM                                           unknown\n             20pM\n\n                          0.2pM\n                                                     concentration\n                    2pM\n                                  0.02pM\n                                           0.002pM -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 98](rendered_slides/Libs_Full-098.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Introduction to the Library Prep Workflow\n                                    NEBNext UltraExpress\u00ae DNA\n\n\n\n\nMax Fritsch | Field Application Scientist -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 99](rendered_slides/Libs_Full-099.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Streamlining the library prep workflow\n\n\n\n\n                                   ONE        ONE\nDNA Fragmentation                [adaptor]   cycle #\n\n                    End Repair   Adaptor      PCR\n                                                        Clean-\n                    dA-Tailing   Ligation     Amplify\n                                                        up\n    Enrichment,\nRNA Fragmentation\n    & cDNA\n    generation -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 100](rendered_slides/Libs_Full-100.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- NEBNext UltraExpress\u00ae DNA Library Prep\n                     Workflow\n\n\n\n5\u2032                                     3\u2032\n3\u2032                                     5\u2032   Fragmentation\n                                            \u2022   Random fragmentation overhangs can be 1-4 bases on the 5\u2019 end, blunt ends or 3\u2019\n                                                overhangs\n                                            \u2022   Incubation time determines fragment size\n5\u2032                3\u2032   5\u2032              3\u2032\n3\u2032                5\u2032   3\u2032              5\u2032\n                                            End repair\n         5\u2032                     3\u2032          \u2022   Enzymatic activity that fills in 5\u2019 overhangs and chews back 3\u2019 overhangs to create\n         3\u2032                     5\u2032\n                                                blunt ends\n\n                                            dA-Tailing\n                                            \u2022   Enzymatic activity to \u2018A\u2019-tail the blunt-ended DNA fragments to prepare them for\n                                                adaptor ligation\n5\u2032              A 3\u2032   5\u2032            A 3\u2032\n3\u2032 A              5\u2032   3\u2032 A            5\u2032\n\n         5\u2032                   A 3\u2032\n         3\u2032 A                   5\u2032 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 101](rendered_slides/Libs_Full-101.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- NEBNext UltraExpress\u00ae DNA Library Prep\n                     Workflow\n\n\n\n\n                                                      Adaptor Ligation\n5\u2032\n3\u2032 A\n                    A 3\u2032\n                      5\u2032\n                           5\u2032\n                           3\u2032 A\n                                               A 3\u2032\n                                                 5\u2032\n                                                      \u2022   Ligation of the T overhang of the hairpin adaptor to the A overhang of the fragmented\n           5\u2032                     A 3\u2032\n                                                          DNA\n           3\u2032 A                       5\u2032\n\n\n\n\n                                                      Hairpin Adaptor\n                                                      \u2022   Higher conversion rates\n                                                      \u2022   Greater complexity\n                                                                                                                   3\u2032\n\n           5\u2032                         3\u2032\n                                                      \u2022   Minimizes adaptor dimer formation                        5\u2032 T\n                                                                                                                                     U\n\n                T                 A\n       U                                   U\n                A                 T\n           3\u2032                         5\u2032 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 102](rendered_slides/Libs_Full-102.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- NEBNext UltraExpress\u00ae DNA Library Prep\n              Workflow\n\n\n\n\n           5\u2032           3\u2032\n                T   A\n       U                     U\n                A   T\n           3\u2032           5\u2032\n\n\n                                             \u2018U\u2019 excision\n                                             \u2022   USER enzyme removes in a series of enzymatic reactions the uracil base to\n                                                 open the NEBNext hairpin adaptor in preparation for PCR\n\n\n                                      USER\n\n\n\n  5\u2032                             3\u2032\n                T   A\n                A   T\n  3\u2032                             5\u2032\n                                       U -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 103](rendered_slides/Libs_Full-103.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- NEBNext UltraExpress\u00ae DNA Library Prep\n                    Workflow\n\n\n\n\n                                                                        5\u2032\n                5\u2032                                      3\u2032\n                          T               A                        P7\n                                                              BC\n                                                   3\u2032\n                     3\u2032\n           BC\n                          A\n                                                                             PCR enrichment\n      P7                                  T\n\n\n 5\u2032\n                3\u2032                                      5\u2032                   \u2022   Amplifies adaptor ligated DNA\n                                                                             \u2022   Adds sample specific Barcode/Index\n                                  st\n                              1        PCR Cycle\n\n\n\n                                                                             Cycle 1: the barcoded P7 PCR primer anneals, but there is still sequence\n                                                                             missing on both strands\n                5\u2032                                           3\u2032\n                3\u2032                                                      5\u2032\n\n\n\n5\u2032                                                      3\u2032\n           3\u2032                                           5\u2032 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 104](rendered_slides/Libs_Full-104.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- NEBNext UltraExpress\u00ae DNA Library Prep\n                    Workflow\n\n\n\n\n      P5   BC\n 5\u2032              3\u2032\n            3\u2032                                        5\u2032\n\n\n\n5\u2032                                         3\u2032              PCR enrichment\n                                3\u2032\n                                            BC   P5\n                                                      5\u2032\n                                                           \u2022   Amplifies adaptor ligated DNA\n                                                           \u2022   Adds sample specific Barcode/Index\n                          nd\n                      2        PCR Cycle\n\n\n\n                                                           Cycle 2: Barcoded P5 primer anneals; the result is one full length strand\n\n 5\u2032                                                   3\u2032\n            3\u2032                                        5\u2032\n\n\n\n5\u2032                                         3\u2032\n3\u2032                                                    5\u2032 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 105](rendered_slides/Libs_Full-105.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- NEBNext UltraExpress\u00ae DNA Library Prep\n                   Workflow\n\n\n\n5\u2032                                   3\u2032\n       3\u2032                            5\u2032\n\n\n\n5\u2032                              3\u2032\n3\u2032                                   5\u2032\n\n                                          PCR enrichment\n                   rd\n                  3 PCR Cycle\n                                          \u2022   Amplifies adaptor ligated DNA\n                                          \u2022   Adds sample specific Barcode/Index\n5\u2032                                   3\u2032\n3\u2032                                   5\u2032\n\n\n5\u2032                                   3\u2032\n                                          Cycle 3+: Both strands are full length; DNA is amplified by PCR\n3\u2032                                   5\u2032\n\n\n5\u2032                                   3\u2032\n3\u2032                                   5\u2032\n\n\n5\u2032                                   3\u2032\n3\u2032                                   5\u2032 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 106](rendered_slides/Libs_Full-106.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- NEBNext UltraExpress\u00ae DNA Library Prep\n                    Workflow\n\n\n\n\n                         T\n                     U\n      P5   BC                               5\u2032\n 5\u2032             3\u2032                                    Library cleanup\n                                       P7\n\n                             3\u2032\n                                  BC                  \u2022   Removes enzymes and buffer from the PCR\n                                                      \u2022   Removes any leftover adaptor dimer\n5\u2032                                               3\u2032\n3\u2032                                               5\u2032   \u2022   Removes PCR primers\n5\u2032                                               3\u2032   \u2022   Phased Clea-up: double clean-up\n3\u2032                                               5\u2032\n\n\n5\u2032                                               3\u2032\n3\u2032                                               5\u2032\n\n\n5\u2032                                               3\u2032\n3\u2032                                               5\u2032 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 107](rendered_slides/Libs_Full-107.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Quick Reminder: Best\n            Practices in the Lab\n1. Thaw the buffers on top of the ice or in your hand and vortex to mix.\n\n2. DO NOT vortex enzymes!\n\n3. Mix enzymes by flicking the tube gently and inverting the tube.\n\n4. Quick spin down the tubes containing the enzymes and buffers.\n\n5. Keep enzymes and thawed buffers on ice.\n\n6. Keep reactions on ice unless it is specified to stay at room temp (RT)\n\n7. Mix reactions thoroughly by pipetting up and down as indicated in\n  protocol. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 108](rendered_slides/Libs_Full-108.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Part Two -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 109](rendered_slides/Libs_Full-109.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Talk Outline\n\n\n\u00bb Libraries for other sequencing\n  platforms.\n\u00bb Types of NGS sequencing library?\n\u00bb Automation of library prep -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 110](rendered_slides/Libs_Full-110.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Sequencers -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 111](rendered_slides/Libs_Full-111.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Sequencers -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 112](rendered_slides/Libs_Full-112.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- PacBio Library Prep -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 113](rendered_slides/Libs_Full-113.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Library\n                                                      Preps\n\n\n                  Fragment DNA\n\n\n\n\n                  Repair Ends\n\n\n\n\nA-Tailing\n\n\n\n\nLigate Adapters                     Ligate Adapters\n\n\n                                                       T        A\n\n                                                       A            T\n                      Exonuclease\n                   digestion of linear\n                          DNA\n\n\n\n\n                        2 x SPRI\n\n\n\n\n                  Quantify and QC -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 114](rendered_slides/Libs_Full-114.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- ONT 1D Library prep -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 115](rendered_slides/Libs_Full-115.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 116](rendered_slides/Libs_Full-116.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Sequencers -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 117](rendered_slides/Libs_Full-117.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Alternative standard library\n       prep methods -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 118](rendered_slides/Libs_Full-118.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Next generation\nenzymatic shearing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 119](rendered_slides/Libs_Full-119.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Ultra II NEB FS DNA:\n      stem\n\n\n             Kit: Enzymatic fragmentation, combined with Ultra II DNA Library Prep reagents\n\n\n                         Fragmentation/                         Clean up/\n                         End Repair/             Adaptor        Size                                       Clean up\n                         dA-Tailing              Ligation       Selection         Amplification\n\n\n\n                                          Hands-on: 8-14 min.    Ultra II DNA FS input amounts:\n                                          Total: 1.5-3 hrs       100 pg \u2013 0.5 \u00b5g, with a single protocol\n\n\n       \uf076 Robust, easy to use and compatible with input DNA in water and standard DNA buffers (TE, Tris-HCl).\n\n       \uf076 Capable of generating a wide range of insert sizes, by varying incubation time.\n\n       \uf076 Uses the same fragmentation protocol, for all input amounts and GC contents.\n\n       \uf076 Produces high yields of high quality libraries.\n\n\n\nUltraII FS. Shearing relatively unaffected by contaminants, GC\ncontent and input amount. Controllable fragment sizes. Can use\nfor low input. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 120](rendered_slides/Libs_Full-120.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 121](rendered_slides/Libs_Full-121.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 122](rendered_slides/Libs_Full-122.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 123](rendered_slides/Libs_Full-123.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 124](rendered_slides/Libs_Full-124.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Nextera -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 125](rendered_slides/Libs_Full-125.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Nextera XT\n\n\u00bb It works but\n  \u00bb Can be expensive\n  \u00bb Size of fragments sensitive to DNA amount,\n    GC content and contaminants\n     \u00bb Small inserts. Adapter. Lower mappability\n     \u00bb DNA not in correct size range\n     \u00bb More variable insert sizes\n     \u00bb More variable coverage when multiplexing -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 126](rendered_slides/Libs_Full-126.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Illumina NextFlex\n          Okay but insert sizes 300bp\n\n\n\n\nNextera Flex. Much much better than old Nextera and XT. Shearing\nrelatively unaffected by contaminants, GC content and input amount.\nExpensive and not too easy to tune fragment sizes. -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 127](rendered_slides/Libs_Full-127.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Other library types -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 128](rendered_slides/Libs_Full-128.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- What is Targeted\n               Sequencing??\n\u201cTargeted resequencing is a variation of re-sequencing where only a\n  small subset of the genome is sequenced, such as the exome, a\n   particular chromosome, a set of genes or a region of interest\u201d -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 129](rendered_slides/Libs_Full-129.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- NoPCR (PCRfree) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 130](rendered_slides/Libs_Full-130.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Amplification bias\n                                                      B.pertussis\n\nTheoretical P.falciparum       No-PCR P.falciparum\n                                                                    Sequence data from both\n                                                                    raw and mapped no-PCR\n                                                                    datasets represent much\n                       standard P.falciparum\n                                                                    closer the base\n                                                                    composition of the\n                                                                    P.falciparum 3D7 genome\n                                                                    than the standard datasets.\n\n\n\n\n                      theoretical GC content\n\n                      raw\n                                  no-PCR libr GC content\n                      mapped\n\n\n\n                      raw\n                                  standard libr GC content\n                      mapped -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 131](rendered_slides/Libs_Full-131.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- No-PCR library preparation :\n                   FP1 and FP2\u2019:part of the adapter sequence\n\n       +\n\n\n                   Enrichment for fully ligated templates during the\n                   cluster generation stage\n                   \uf0fcNon-ligated templates cannot attach to the flowcell\n                   \uf0fcSemi-ligated templates cannot form a bridge and thus\n                   clusters\n\n\n\n\n                  More crucial than for the standard library\n                  preparation are:\n                  \uf0fcSample DNA quality and quantity\n                  \uf0fcQuantification of libraries with lower concentration -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 132](rendered_slides/Libs_Full-132.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- RNA seq -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 133](rendered_slides/Libs_Full-133.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- RNA seq Approaches\n\n\n\u00bb 3\u2019 gene expression via oligo dT priming\n\u00bb Full length cDNA\n\u00bb Single cell 3\u2019, 5\u2019 or probe based\n\n\u00bb Enrichment and rRNA depletion\n  \u00bb Typically oligodT\n  \u00bb Probe based for globin, RNA or other\n    depletion -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 134](rendered_slides/Libs_Full-134.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- RNA seq\n               Illumina cDNA protocol\n                                            RNA\n                                            mRNA\n                                            Fragment - metal ions\nneeds good quality DNA free RNA             cDNA synthesis\nIdeally RIN > 8                             cDNA\n                                            End-repair\n\n                                            A-Tail\n                                            Ligate to adapter\n\n                                            Size select/Gel purify\n\n                                            PCR\n\n                                            Quantitate + Characterise -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 135](rendered_slides/Libs_Full-135.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Directional RNA seq\n                  dUTP/USER protocol\n\n\n\nParkhomchuk et al.,\nNAR. 2009, 37 (18) -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 136](rendered_slides/Libs_Full-136.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- small RNA\n\n                              RNA\n                              miRNA\n                              Ligate 3\u2019 adapter\n                              Ligate 5\u2019 adapter\n                              RT\n                              PCR\n                              Size select/Gel purify\n                              Quantitate + Characterise\n\n\n\u00bb Notes\n-need 2ug total RNA -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 137](rendered_slides/Libs_Full-137.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- ChIP\nChromatin Immunoprecipitation -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 138](rendered_slides/Libs_Full-138.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 139](rendered_slides/Libs_Full-139.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- HiC -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 140](rendered_slides/Libs_Full-140.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- ATAC and DNAse Seq -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 141](rendered_slides/Libs_Full-141.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Methyl Seq -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 142](rendered_slides/Libs_Full-142.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- NEB EM-Seq -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 143](rendered_slides/Libs_Full-143.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- How Many Libraries? -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 144](rendered_slides/Libs_Full-144.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Sequence yield\n\n\n                       9000Gb\n\n               500Gb\n\n\n       100Gb\n\n\n\n1Gb -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 145](rendered_slides/Libs_Full-145.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Yellow Answers\n                To make all these LIBRARIES is\n\n\n\n             lot of work\nSamples from different sources\nDifferent levels of complexity\nLarge sample sets -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 146](rendered_slides/Libs_Full-146.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Headache!! -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 147](rendered_slides/Libs_Full-147.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Automation -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 148](rendered_slides/Libs_Full-148.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Approaches to Automation -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 149](rendered_slides/Libs_Full-149.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- High throughput manual -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 150](rendered_slides/Libs_Full-150.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Low throughput automation\nAgilent Magnis \u2013 8 sample automation -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 151](rendered_slides/Libs_Full-151.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Low throughput automation\n\n\n\n                Apollo 324\nhttp://www.wafergen.com/products/apollo-324\n\n                                                        Tecan MagicPrep NGS\n                                                    http://www.tecan.com/magic\n\n\n\n\n                Agilent Magnis\n           http://www.agilent.com             PE BioCule -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 152](rendered_slides/Libs_Full-152.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- High throughput automation\n\n\n  Hamilton Star\n                               Tecan fluent\n\n\n\n\n                                              Agilent Bravo\n                  Revvity Sciclone NGS\n Beckman i7 -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 153](rendered_slides/Libs_Full-153.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Introductory systems?\n\n\n\n\nOpentrons flex        SPT Firefly -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 154](rendered_slides/Libs_Full-154.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 155](rendered_slides/Libs_Full-155.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!--  -->"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "![Slide 156](rendered_slides/Libs_Full-156.jpg)\n",
    "\n",
    "<!-- SEARCHABLE TEXT CONTENT -->\n",
    "<!-- Any Questions ?\n\n   mq1@sanger.ac.uk -->"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "name": "python",
   "version": "3.8.10"
  },
  "celltoolbar": "Slideshow"
 },
 "nbformat": 4,
 "nbformat_minor": 4
}